2025-09-25 07:05:28,727 1416 INFO ? odoo.modules.loading: loading 1 modules... 
2025-09-25 07:05:28,733 1416 INFO ? odoo.modules.loading: 1 modules loaded in 0.01s, 0 queries (+0 extra) 
2025-09-25 07:05:28,739 1416 INFO ? odoo.modules.loading: loading 61 modules... 
2025-09-25 07:05:28,775 1416 INFO ? odoo.modules.loading: Loading module stock_qr_tracking (60/61) 
2025-09-25 07:05:28,993 1416 INFO ? odoo.modules.registry: module stock_qr_tracking: creating or updating database tables 
2025-09-25 07:05:29,020 1416 INFO ? odoo.modules.loading: loading stock_qr_tracking/security/ir.model.access.csv 
2025-09-25 07:05:29,044 1416 INFO ? odoo.modules.loading: loading stock_qr_tracking/views/stock_move_line_views.xml 
2025-09-25 07:05:29,047 1416 WARNING ? odoo.modules.loading: Transient module states were reset 
2025-09-25 07:05:29,047 1416 ERROR ? odoo.modules.registry: Failed to load registry 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 87, in __new__
    return cls.registries[db_name]
           ~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\lru.py", line 34, in __getitem__
    a = self.d[obj]
        ~~~~~~^^^^^
KeyError: 'ssi_odoo_demo'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\cache.py", line 99, in lookup
    r = d[key]
        ~^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\lru.py", line 34, in __getitem__
    a = self.d[obj]
        ~~~~~~^^^^^
KeyError: ('ir.model.data', <function IrModelData._xmlid_lookup at 0x000001AA6BB7F7E0>, 'stock.stock_move_line_view_tree')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 564, in _tag_root
    f(rec)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 429, in _tag_record
    f_val = self.id_get(f_ref, raise_if_not_found=nodeattr2bool(rec, 'forcecreate', True))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 546, in id_get
    res = self.model_id_get(id_str, raise_if_not_found)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 552, in model_id_get
    return self.env['ir.model.data']._xmlid_to_res_model_res_id(id_str, raise_if_not_found=raise_if_not_found)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\addons\base\models\ir_model.py", line 2205, in _xmlid_to_res_model_res_id
    return self._xmlid_lookup(xmlid)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\cache.py", line 104, in lookup
    value = d[key] = self.method(*args, **kwargs)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\addons\base\models\ir_model.py", line 2198, in _xmlid_lookup
    raise ValueError('External ID not found in the system: %s' % xmlid)
ValueError: External ID not found in the system: stock.stock_move_line_view_tree

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 577, in _tag_root
    raise ParseError('while parsing %s:%s, somewhere inside\n%s' % (
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:4, somewhere inside
<record id="view_picking_move_line_tree_inherit_qr" model="ir.ui.view">
        <field name="name">stock.picking.move.line.tree.inherit.qr</field>
        <field name="model">stock.move.line</field>
        <!-- PERBAIKAN: Menggunakan ID view yang benar untuk Odoo 17 -->
        <field name="inherit_id" ref="stock.stock_move_line_view_tree"/>
        <field name="arch" type="xml">
            <xpath expr="//tree" position="inside">
                <field name="qr_code" widget="image" readonly="1"/>
                <field name="is_verified"/>
            </xpath>
        </field>
    </record>
2025-09-25 07:05:29,050 1416 ERROR ? odoo.http: Exception during request handling. 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 87, in __new__
    return cls.registries[db_name]
           ~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\lru.py", line 34, in __getitem__
    a = self.d[obj]
        ~~~~~~^^^^^
KeyError: 'ssi_odoo_demo'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\cache.py", line 99, in lookup
    r = d[key]
        ~^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\lru.py", line 34, in __getitem__
    a = self.d[obj]
        ~~~~~~^^^^^
KeyError: ('ir.model.data', <function IrModelData._xmlid_lookup at 0x000001AA6BB7F7E0>, 'stock.stock_move_line_view_tree')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 564, in _tag_root
    f(rec)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 429, in _tag_record
    f_val = self.id_get(f_ref, raise_if_not_found=nodeattr2bool(rec, 'forcecreate', True))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 546, in id_get
    res = self.model_id_get(id_str, raise_if_not_found)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 552, in model_id_get
    return self.env['ir.model.data']._xmlid_to_res_model_res_id(id_str, raise_if_not_found=raise_if_not_found)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\addons\base\models\ir_model.py", line 2205, in _xmlid_to_res_model_res_id
    return self._xmlid_lookup(xmlid)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\cache.py", line 104, in lookup
    value = d[key] = self.method(*args, **kwargs)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\addons\base\models\ir_model.py", line 2198, in _xmlid_lookup
    raise ValueError('External ID not found in the system: %s' % xmlid)
ValueError: External ID not found in the system: stock.stock_move_line_view_tree

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 2396, in __call__
    response = request._serve_db()
               ^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 1949, in _serve_db
    self.registry = Registry(self.db).check_signaling()
                    ^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 89, in __new__
    return cls.new(db_name)
           ^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 577, in _tag_root
    raise ParseError('while parsing %s:%s, somewhere inside\n%s' % (
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:4, somewhere inside
<record id="view_picking_move_line_tree_inherit_qr" model="ir.ui.view">
        <field name="name">stock.picking.move.line.tree.inherit.qr</field>
        <field name="model">stock.move.line</field>
        <!-- PERBAIKAN: Menggunakan ID view yang benar untuk Odoo 17 -->
        <field name="inherit_id" ref="stock.stock_move_line_view_tree"/>
        <field name="arch" type="xml">
            <xpath expr="//tree" position="inside">
                <field name="qr_code" widget="image" readonly="1"/>
                <field name="is_verified"/>
            </xpath>
        </field>
    </record>
2025-09-25 07:05:29,053 1416 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:05:29] "GET /websocket?version=17.0-1 HTTP/1.1" 500 - 64 0.031 0.300
2025-09-25 07:05:31,272 1416 INFO ? odoo.modules.loading: loading 1 modules... 
2025-09-25 07:05:31,277 1416 INFO ? odoo.modules.loading: 1 modules loaded in 0.01s, 0 queries (+0 extra) 
2025-09-25 07:05:31,284 1416 INFO ? odoo.modules.loading: loading 61 modules... 
2025-09-25 07:05:31,324 1416 INFO ? odoo.modules.loading: Loading module stock_qr_tracking (60/61) 
2025-09-25 07:05:31,616 1416 INFO ? odoo.modules.registry: module stock_qr_tracking: creating or updating database tables 
2025-09-25 07:05:31,643 1416 INFO ? odoo.modules.loading: loading stock_qr_tracking/security/ir.model.access.csv 
2025-09-25 07:05:31,669 1416 INFO ? odoo.modules.loading: loading stock_qr_tracking/views/stock_move_line_views.xml 
2025-09-25 07:05:31,673 1416 WARNING ? odoo.modules.loading: Transient module states were reset 
2025-09-25 07:05:31,673 1416 ERROR ? odoo.modules.registry: Failed to load registry 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 87, in __new__
    return cls.registries[db_name]
           ~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\lru.py", line 34, in __getitem__
    a = self.d[obj]
        ~~~~~~^^^^^
KeyError: 'ssi_odoo_demo'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\cache.py", line 99, in lookup
    r = d[key]
        ~^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\lru.py", line 34, in __getitem__
    a = self.d[obj]
        ~~~~~~^^^^^
KeyError: ('ir.model.data', <function IrModelData._xmlid_lookup at 0x000001AA6BB7F7E0>, 'stock.stock_move_line_view_tree')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 564, in _tag_root
    f(rec)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 429, in _tag_record
    f_val = self.id_get(f_ref, raise_if_not_found=nodeattr2bool(rec, 'forcecreate', True))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 546, in id_get
    res = self.model_id_get(id_str, raise_if_not_found)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 552, in model_id_get
    return self.env['ir.model.data']._xmlid_to_res_model_res_id(id_str, raise_if_not_found=raise_if_not_found)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\addons\base\models\ir_model.py", line 2205, in _xmlid_to_res_model_res_id
    return self._xmlid_lookup(xmlid)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\cache.py", line 104, in lookup
    value = d[key] = self.method(*args, **kwargs)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\addons\base\models\ir_model.py", line 2198, in _xmlid_lookup
    raise ValueError('External ID not found in the system: %s' % xmlid)
ValueError: External ID not found in the system: stock.stock_move_line_view_tree

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 577, in _tag_root
    raise ParseError('while parsing %s:%s, somewhere inside\n%s' % (
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:4, somewhere inside
<record id="view_picking_move_line_tree_inherit_qr" model="ir.ui.view">
        <field name="name">stock.picking.move.line.tree.inherit.qr</field>
        <field name="model">stock.move.line</field>
        <!-- PERBAIKAN: Menggunakan ID view yang benar untuk Odoo 17 -->
        <field name="inherit_id" ref="stock.stock_move_line_view_tree"/>
        <field name="arch" type="xml">
            <xpath expr="//tree" position="inside">
                <field name="qr_code" widget="image" readonly="1"/>
                <field name="is_verified"/>
            </xpath>
        </field>
    </record>
2025-09-25 07:05:31,675 1416 ERROR ? odoo.http: Exception during request handling. 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 87, in __new__
    return cls.registries[db_name]
           ~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\lru.py", line 34, in __getitem__
    a = self.d[obj]
        ~~~~~~^^^^^
KeyError: 'ssi_odoo_demo'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\cache.py", line 99, in lookup
    r = d[key]
        ~^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\lru.py", line 34, in __getitem__
    a = self.d[obj]
        ~~~~~~^^^^^
KeyError: ('ir.model.data', <function IrModelData._xmlid_lookup at 0x000001AA6BB7F7E0>, 'stock.stock_move_line_view_tree')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 564, in _tag_root
    f(rec)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 429, in _tag_record
    f_val = self.id_get(f_ref, raise_if_not_found=nodeattr2bool(rec, 'forcecreate', True))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 546, in id_get
    res = self.model_id_get(id_str, raise_if_not_found)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 552, in model_id_get
    return self.env['ir.model.data']._xmlid_to_res_model_res_id(id_str, raise_if_not_found=raise_if_not_found)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\addons\base\models\ir_model.py", line 2205, in _xmlid_to_res_model_res_id
    return self._xmlid_lookup(xmlid)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\cache.py", line 104, in lookup
    value = d[key] = self.method(*args, **kwargs)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\addons\base\models\ir_model.py", line 2198, in _xmlid_lookup
    raise ValueError('External ID not found in the system: %s' % xmlid)
ValueError: External ID not found in the system: stock.stock_move_line_view_tree

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 2396, in __call__
    response = request._serve_db()
               ^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 1949, in _serve_db
    self.registry = Registry(self.db).check_signaling()
                    ^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 89, in __new__
    return cls.new(db_name)
           ^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 577, in _tag_root
    raise ParseError('while parsing %s:%s, somewhere inside\n%s' % (
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:4, somewhere inside
<record id="view_picking_move_line_tree_inherit_qr" model="ir.ui.view">
        <field name="name">stock.picking.move.line.tree.inherit.qr</field>
        <field name="model">stock.move.line</field>
        <!-- PERBAIKAN: Menggunakan ID view yang benar untuk Odoo 17 -->
        <field name="inherit_id" ref="stock.stock_move_line_view_tree"/>
        <field name="arch" type="xml">
            <xpath expr="//tree" position="inside">
                <field name="qr_code" widget="image" readonly="1"/>
                <field name="is_verified"/>
            </xpath>
        </field>
    </record>
2025-09-25 07:05:31,677 1416 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:05:31] "GET /web?debug=1 HTTP/1.1" 500 - 64 0.032 0.379
2025-09-25 07:05:33,269 1416 INFO ? odoo.modules.loading: loading 1 modules... 
2025-09-25 07:05:33,276 1416 INFO ? odoo.modules.loading: 1 modules loaded in 0.01s, 0 queries (+0 extra) 
2025-09-25 07:05:33,283 1416 INFO ? odoo.modules.loading: loading 61 modules... 
2025-09-25 07:05:33,317 1416 INFO ? odoo.modules.loading: Loading module stock_qr_tracking (60/61) 
2025-09-25 07:05:33,534 1416 INFO ? odoo.modules.registry: module stock_qr_tracking: creating or updating database tables 
2025-09-25 07:05:33,565 1416 INFO ? odoo.modules.loading: loading stock_qr_tracking/security/ir.model.access.csv 
2025-09-25 07:05:33,591 1416 INFO ? odoo.modules.loading: loading stock_qr_tracking/views/stock_move_line_views.xml 
2025-09-25 07:05:33,596 1416 WARNING ? odoo.modules.loading: Transient module states were reset 
2025-09-25 07:05:33,596 1416 ERROR ? odoo.modules.registry: Failed to load registry 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 87, in __new__
    return cls.registries[db_name]
           ~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\lru.py", line 34, in __getitem__
    a = self.d[obj]
        ~~~~~~^^^^^
KeyError: 'ssi_odoo_demo'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\cache.py", line 99, in lookup
    r = d[key]
        ~^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\lru.py", line 34, in __getitem__
    a = self.d[obj]
        ~~~~~~^^^^^
KeyError: ('ir.model.data', <function IrModelData._xmlid_lookup at 0x000001AA6BB7F7E0>, 'stock.stock_move_line_view_tree')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 564, in _tag_root
    f(rec)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 429, in _tag_record
    f_val = self.id_get(f_ref, raise_if_not_found=nodeattr2bool(rec, 'forcecreate', True))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 546, in id_get
    res = self.model_id_get(id_str, raise_if_not_found)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 552, in model_id_get
    return self.env['ir.model.data']._xmlid_to_res_model_res_id(id_str, raise_if_not_found=raise_if_not_found)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\addons\base\models\ir_model.py", line 2205, in _xmlid_to_res_model_res_id
    return self._xmlid_lookup(xmlid)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\cache.py", line 104, in lookup
    value = d[key] = self.method(*args, **kwargs)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\addons\base\models\ir_model.py", line 2198, in _xmlid_lookup
    raise ValueError('External ID not found in the system: %s' % xmlid)
ValueError: External ID not found in the system: stock.stock_move_line_view_tree

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 577, in _tag_root
    raise ParseError('while parsing %s:%s, somewhere inside\n%s' % (
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:4, somewhere inside
<record id="view_picking_move_line_tree_inherit_qr" model="ir.ui.view">
        <field name="name">stock.picking.move.line.tree.inherit.qr</field>
        <field name="model">stock.move.line</field>
        <!-- PERBAIKAN: Menggunakan ID view yang benar untuk Odoo 17 -->
        <field name="inherit_id" ref="stock.stock_move_line_view_tree"/>
        <field name="arch" type="xml">
            <xpath expr="//tree" position="inside">
                <field name="qr_code" widget="image" readonly="1"/>
                <field name="is_verified"/>
            </xpath>
        </field>
    </record>
2025-09-25 07:05:33,598 1416 ERROR ? odoo.http: Exception during request handling. 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 87, in __new__
    return cls.registries[db_name]
           ~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\lru.py", line 34, in __getitem__
    a = self.d[obj]
        ~~~~~~^^^^^
KeyError: 'ssi_odoo_demo'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\cache.py", line 99, in lookup
    r = d[key]
        ~^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\lru.py", line 34, in __getitem__
    a = self.d[obj]
        ~~~~~~^^^^^
KeyError: ('ir.model.data', <function IrModelData._xmlid_lookup at 0x000001AA6BB7F7E0>, 'stock.stock_move_line_view_tree')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 564, in _tag_root
    f(rec)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 429, in _tag_record
    f_val = self.id_get(f_ref, raise_if_not_found=nodeattr2bool(rec, 'forcecreate', True))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 546, in id_get
    res = self.model_id_get(id_str, raise_if_not_found)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 552, in model_id_get
    return self.env['ir.model.data']._xmlid_to_res_model_res_id(id_str, raise_if_not_found=raise_if_not_found)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\addons\base\models\ir_model.py", line 2205, in _xmlid_to_res_model_res_id
    return self._xmlid_lookup(xmlid)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\cache.py", line 104, in lookup
    value = d[key] = self.method(*args, **kwargs)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\addons\base\models\ir_model.py", line 2198, in _xmlid_lookup
    raise ValueError('External ID not found in the system: %s' % xmlid)
ValueError: External ID not found in the system: stock.stock_move_line_view_tree

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 2396, in __call__
    response = request._serve_db()
               ^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 1949, in _serve_db
    self.registry = Registry(self.db).check_signaling()
                    ^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 89, in __new__
    return cls.new(db_name)
           ^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 577, in _tag_root
    raise ParseError('while parsing %s:%s, somewhere inside\n%s' % (
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:4, somewhere inside
<record id="view_picking_move_line_tree_inherit_qr" model="ir.ui.view">
        <field name="name">stock.picking.move.line.tree.inherit.qr</field>
        <field name="model">stock.move.line</field>
        <!-- PERBAIKAN: Menggunakan ID view yang benar untuk Odoo 17 -->
        <field name="inherit_id" ref="stock.stock_move_line_view_tree"/>
        <field name="arch" type="xml">
            <xpath expr="//tree" position="inside">
                <field name="qr_code" widget="image" readonly="1"/>
                <field name="is_verified"/>
            </xpath>
        </field>
    </record>
2025-09-25 07:05:33,601 1416 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:05:33] "GET /web/service-worker.js HTTP/1.1" 500 - 64 0.040 0.302
2025-09-25 07:06:29,834 1416 INFO ? odoo.modules.loading: loading 1 modules... 
2025-09-25 07:06:29,843 1416 INFO ? odoo.modules.loading: 1 modules loaded in 0.01s, 0 queries (+0 extra) 
2025-09-25 07:06:29,853 1416 INFO ? odoo.modules.loading: loading 61 modules... 
2025-09-25 07:06:29,917 1416 INFO ? odoo.modules.loading: Loading module stock_qr_tracking (60/61) 
2025-09-25 07:06:30,350 1416 INFO ? odoo.modules.registry: module stock_qr_tracking: creating or updating database tables 
2025-09-25 07:06:30,386 1416 INFO ? odoo.modules.loading: loading stock_qr_tracking/security/ir.model.access.csv 
2025-09-25 07:06:30,427 1416 INFO ? odoo.modules.loading: loading stock_qr_tracking/views/stock_move_line_views.xml 
2025-09-25 07:06:30,432 1416 WARNING ? odoo.modules.loading: Transient module states were reset 
2025-09-25 07:06:30,433 1416 ERROR ? odoo.modules.registry: Failed to load registry 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 87, in __new__
    return cls.registries[db_name]
           ~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\lru.py", line 34, in __getitem__
    a = self.d[obj]
        ~~~~~~^^^^^
KeyError: 'ssi_odoo_demo'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\cache.py", line 99, in lookup
    r = d[key]
        ~^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\lru.py", line 34, in __getitem__
    a = self.d[obj]
        ~~~~~~^^^^^
KeyError: ('ir.model.data', <function IrModelData._xmlid_lookup at 0x000001AA6BB7F7E0>, 'stock.stock_move_line_view_tree')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 564, in _tag_root
    f(rec)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 429, in _tag_record
    f_val = self.id_get(f_ref, raise_if_not_found=nodeattr2bool(rec, 'forcecreate', True))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 546, in id_get
    res = self.model_id_get(id_str, raise_if_not_found)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 552, in model_id_get
    return self.env['ir.model.data']._xmlid_to_res_model_res_id(id_str, raise_if_not_found=raise_if_not_found)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\addons\base\models\ir_model.py", line 2205, in _xmlid_to_res_model_res_id
    return self._xmlid_lookup(xmlid)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\cache.py", line 104, in lookup
    value = d[key] = self.method(*args, **kwargs)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\addons\base\models\ir_model.py", line 2198, in _xmlid_lookup
    raise ValueError('External ID not found in the system: %s' % xmlid)
ValueError: External ID not found in the system: stock.stock_move_line_view_tree

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 577, in _tag_root
    raise ParseError('while parsing %s:%s, somewhere inside\n%s' % (
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:3, somewhere inside
<record id="view_picking_move_line_tree_inherit_qr" model="ir.ui.view">
        <field name="name">stock.picking.move.line.tree.inherit.qr</field>
        <field name="model">stock.move.line</field>
        <field name="inherit_id" ref="stock.stock_move_line_view_tree"/>
        <field name="arch" type="xml">
            <xpath expr="//tree" position="inside">
                <field name="qr_code" widget="image" readonly="1"/>
                <field name="is_verified"/>
            </xpath>
        </field>
    </record>
2025-09-25 07:06:30,436 1416 ERROR ? odoo.http: Exception during request handling. 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 87, in __new__
    return cls.registries[db_name]
           ~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\lru.py", line 34, in __getitem__
    a = self.d[obj]
        ~~~~~~^^^^^
KeyError: 'ssi_odoo_demo'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\cache.py", line 99, in lookup
    r = d[key]
        ~^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\lru.py", line 34, in __getitem__
    a = self.d[obj]
        ~~~~~~^^^^^
KeyError: ('ir.model.data', <function IrModelData._xmlid_lookup at 0x000001AA6BB7F7E0>, 'stock.stock_move_line_view_tree')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 564, in _tag_root
    f(rec)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 429, in _tag_record
    f_val = self.id_get(f_ref, raise_if_not_found=nodeattr2bool(rec, 'forcecreate', True))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 546, in id_get
    res = self.model_id_get(id_str, raise_if_not_found)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 552, in model_id_get
    return self.env['ir.model.data']._xmlid_to_res_model_res_id(id_str, raise_if_not_found=raise_if_not_found)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\addons\base\models\ir_model.py", line 2205, in _xmlid_to_res_model_res_id
    return self._xmlid_lookup(xmlid)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\cache.py", line 104, in lookup
    value = d[key] = self.method(*args, **kwargs)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\addons\base\models\ir_model.py", line 2198, in _xmlid_lookup
    raise ValueError('External ID not found in the system: %s' % xmlid)
ValueError: External ID not found in the system: stock.stock_move_line_view_tree

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 2396, in __call__
    response = request._serve_db()
               ^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 1949, in _serve_db
    self.registry = Registry(self.db).check_signaling()
                    ^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 89, in __new__
    return cls.new(db_name)
           ^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 577, in _tag_root
    raise ParseError('while parsing %s:%s, somewhere inside\n%s' % (
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:3, somewhere inside
<record id="view_picking_move_line_tree_inherit_qr" model="ir.ui.view">
        <field name="name">stock.picking.move.line.tree.inherit.qr</field>
        <field name="model">stock.move.line</field>
        <field name="inherit_id" ref="stock.stock_move_line_view_tree"/>
        <field name="arch" type="xml">
            <xpath expr="//tree" position="inside">
                <field name="qr_code" widget="image" readonly="1"/>
                <field name="is_verified"/>
            </xpath>
        </field>
    </record>
2025-09-25 07:06:30,441 1416 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:06:30] "GET /websocket?version=17.0-1 HTTP/1.1" 500 - 64 0.037 0.575
2025-09-25 07:07:30,946 1416 INFO ? odoo.modules.loading: loading 1 modules... 
2025-09-25 07:07:30,952 1416 INFO ? odoo.modules.loading: 1 modules loaded in 0.01s, 0 queries (+0 extra) 
2025-09-25 07:07:30,959 1416 INFO ? odoo.modules.loading: loading 61 modules... 
2025-09-25 07:07:30,996 1416 INFO ? odoo.modules.loading: Loading module stock_qr_tracking (60/61) 
2025-09-25 07:07:31,198 1416 INFO ? odoo.modules.registry: module stock_qr_tracking: creating or updating database tables 
2025-09-25 07:07:31,226 1416 INFO ? odoo.modules.loading: loading stock_qr_tracking/security/ir.model.access.csv 
2025-09-25 07:07:31,247 1416 INFO ? odoo.modules.loading: loading stock_qr_tracking/views/stock_move_line_views.xml 
2025-09-25 07:07:31,252 1416 WARNING ? odoo.modules.loading: Transient module states were reset 
2025-09-25 07:07:31,252 1416 ERROR ? odoo.modules.registry: Failed to load registry 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 87, in __new__
    return cls.registries[db_name]
           ~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\lru.py", line 34, in __getitem__
    a = self.d[obj]
        ~~~~~~^^^^^
KeyError: 'ssi_odoo_demo'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\cache.py", line 99, in lookup
    r = d[key]
        ~^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\lru.py", line 34, in __getitem__
    a = self.d[obj]
        ~~~~~~^^^^^
KeyError: ('ir.model.data', <function IrModelData._xmlid_lookup at 0x000001AA6BB7F7E0>, 'stock.stock_move_line_view_tree')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 564, in _tag_root
    f(rec)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 429, in _tag_record
    f_val = self.id_get(f_ref, raise_if_not_found=nodeattr2bool(rec, 'forcecreate', True))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 546, in id_get
    res = self.model_id_get(id_str, raise_if_not_found)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 552, in model_id_get
    return self.env['ir.model.data']._xmlid_to_res_model_res_id(id_str, raise_if_not_found=raise_if_not_found)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\addons\base\models\ir_model.py", line 2205, in _xmlid_to_res_model_res_id
    return self._xmlid_lookup(xmlid)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\cache.py", line 104, in lookup
    value = d[key] = self.method(*args, **kwargs)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\addons\base\models\ir_model.py", line 2198, in _xmlid_lookup
    raise ValueError('External ID not found in the system: %s' % xmlid)
ValueError: External ID not found in the system: stock.stock_move_line_view_tree

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 577, in _tag_root
    raise ParseError('while parsing %s:%s, somewhere inside\n%s' % (
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:3, somewhere inside
<record id="view_picking_move_line_tree_inherit_qr" model="ir.ui.view">
        <field name="name">stock.picking.move.line.tree.inherit.qr</field>
        <field name="model">stock.move.line</field>
        <field name="inherit_id" ref="stock.stock_move_line_view_tree"/>
        <field name="arch" type="xml">
            <xpath expr="//tree" position="inside">
                <field name="qr_code" widget="image" readonly="1"/>
                <field name="is_verified"/>
            </xpath>
        </field>
    </record>
2025-09-25 07:07:31,254 1416 ERROR ? odoo.http: Exception during request handling. 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 87, in __new__
    return cls.registries[db_name]
           ~~~~~~~~~~~~~~^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\lru.py", line 34, in __getitem__
    a = self.d[obj]
        ~~~~~~^^^^^
KeyError: 'ssi_odoo_demo'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\cache.py", line 99, in lookup
    r = d[key]
        ~^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\lru.py", line 34, in __getitem__
    a = self.d[obj]
        ~~~~~~^^^^^
KeyError: ('ir.model.data', <function IrModelData._xmlid_lookup at 0x000001AA6BB7F7E0>, 'stock.stock_move_line_view_tree')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 564, in _tag_root
    f(rec)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 429, in _tag_record
    f_val = self.id_get(f_ref, raise_if_not_found=nodeattr2bool(rec, 'forcecreate', True))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 546, in id_get
    res = self.model_id_get(id_str, raise_if_not_found)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 552, in model_id_get
    return self.env['ir.model.data']._xmlid_to_res_model_res_id(id_str, raise_if_not_found=raise_if_not_found)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\addons\base\models\ir_model.py", line 2205, in _xmlid_to_res_model_res_id
    return self._xmlid_lookup(xmlid)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\cache.py", line 104, in lookup
    value = d[key] = self.method(*args, **kwargs)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\addons\base\models\ir_model.py", line 2198, in _xmlid_lookup
    raise ValueError('External ID not found in the system: %s' % xmlid)
ValueError: External ID not found in the system: stock.stock_move_line_view_tree

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 2396, in __call__
    response = request._serve_db()
               ^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 1949, in _serve_db
    self.registry = Registry(self.db).check_signaling()
                    ^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 89, in __new__
    return cls.new(db_name)
           ^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 577, in _tag_root
    raise ParseError('while parsing %s:%s, somewhere inside\n%s' % (
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:3, somewhere inside
<record id="view_picking_move_line_tree_inherit_qr" model="ir.ui.view">
        <field name="name">stock.picking.move.line.tree.inherit.qr</field>
        <field name="model">stock.move.line</field>
        <field name="inherit_id" ref="stock.stock_move_line_view_tree"/>
        <field name="arch" type="xml">
            <xpath expr="//tree" position="inside">
                <field name="qr_code" widget="image" readonly="1"/>
                <field name="is_verified"/>
            </xpath>
        </field>
    </record>
2025-09-25 07:07:31,257 1416 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:07:31] "GET /websocket?version=17.0-1 HTTP/1.1" 500 - 64 0.035 0.282
2025-09-25 07:08:14,909 1416 INFO ssi_odoo_demo odoo.service.server: Initiating shutdown 
2025-09-25 07:08:14,909 1416 INFO ssi_odoo_demo odoo.service.server: Hit CTRL-C again or send a second signal to force the shutdown. 
2025-09-25 07:08:15,388 1416 INFO ssi_odoo_demo odoo.sql_db: ConnectionPool(used=0/count=0/max=64): Closed 3 connections  
2025-09-25 07:08:18,764 16948 INFO ? odoo: Odoo version 17.0 
2025-09-25 07:08:18,764 16948 INFO ? odoo: Using configuration file at D:\work-project\ssi\odoo_demo\odoo\odoo.conf 
2025-09-25 07:08:18,764 16948 INFO ? odoo: addons paths: ['D:\\work-project\\ssi\\odoo_demo\\odoo\\odoo\\addons', 'd:\\work-project\\ssi\\odoo_demo\\odoo\\data\\addons\\17.0', 'd:\\work-project\\ssi\\odoo_demo\\odoo\\addons', 'd:\\work-project\\ssi\\odoo_demo\\odoo\\custom_addons', 'd:\\work-project\\ssi\\odoo_demo\\odoo\\odoo\\addons'] 
2025-09-25 07:08:18,765 16948 INFO ? odoo: database: odoo@localhost:5432 
2025-09-25 07:08:18,974 16948 INFO ? odoo.addons.base.models.ir_actions_report: Will use the Wkhtmltopdf binary at C:\Program Files\wkhtmltopdf\bin\wkhtmltopdf.exe 
2025-09-25 07:08:19,299 16948 WARNING ? odoo.service.server: 'watchdog' module not installed. Code autoreload feature is disabled 
2025-09-25 07:08:19,303 16948 INFO ? odoo.service.server: HTTP service (werkzeug) running on LAPTOP-O0EP5QSP:8069 
2025-09-25 07:08:19,361 16948 INFO ssi_odoo_demo odoo.modules.loading: loading 1 modules... 
2025-09-25 07:08:19,364 16948 INFO ssi_odoo_demo odoo.modules.loading: 1 modules loaded in 0.00s, 0 queries (+0 extra) 
2025-09-25 07:08:19,385 16948 INFO ssi_odoo_demo odoo.modules.loading: updating modules list 
2025-09-25 07:08:19,389 16948 INFO ssi_odoo_demo odoo.addons.base.models.ir_module: ALLOW access to module.update_list on [] to user __system__ #1 via n/a 
2025-09-25 07:08:20,304 16948 WARNING ssi_odoo_demo odoo.modules.module: Missing `license` key in manifest for 'product_master', defaulting to LGPL-3 
2025-09-25 07:08:20,436 16948 WARNING ssi_odoo_demo odoo.modules.module: Missing `license` key in manifest for 'stock_qr_tracking', defaulting to LGPL-3 
2025-09-25 07:08:20,651 16948 INFO ssi_odoo_demo odoo.addons.base.models.ir_module: ALLOW access to module.button_upgrade on ['Stock QR Tracking'] to user __system__ #1 via n/a 
2025-09-25 07:08:20,651 16948 INFO ssi_odoo_demo odoo.addons.base.models.ir_module: ALLOW access to module.update_list on ['Stock QR Tracking'] to user __system__ #1 via n/a 
2025-09-25 07:08:21,319 16948 INFO ssi_odoo_demo odoo.addons.base.models.ir_module: ALLOW access to module.button_install on [] to user __system__ #1 via n/a 
2025-09-25 07:08:21,351 16948 INFO ssi_odoo_demo odoo.modules.loading: loading 61 modules... 
2025-09-25 07:08:22,305 16948 WARNING ssi_odoo_demo odoo.api.create: The model odoo.addons.product_master.models.product_template is not overriding the create method in batch 
2025-09-25 07:08:22,342 16948 INFO ssi_odoo_demo odoo.modules.loading: Loading module stock_qr_tracking (60/61) 
2025-09-25 07:08:22,650 16948 INFO ssi_odoo_demo odoo.modules.registry: module stock_qr_tracking: creating or updating database tables 
2025-09-25 07:08:22,752 16948 INFO ssi_odoo_demo odoo.modules.loading: loading stock_qr_tracking/security/ir.model.access.csv 
2025-09-25 07:08:22,772 16948 INFO ssi_odoo_demo odoo.modules.loading: loading stock_qr_tracking/views/stock_move_line_views.xml 
2025-09-25 07:08:22,803 16948 WARNING ssi_odoo_demo odoo.addons.base.models.ir_ui_view: Could not find view arch definition in file 'd:\work-project\ssi\odoo_demo\odoo\custom_addons\stock_qr_tracking\views\stock_move_line_views.xml' for xmlid 'stock_qr_tracking.view_move_line_tree_inherit_qr' 
2025-09-25 07:08:22,849 16948 INFO ssi_odoo_demo odoo.modules.loading: Module stock_qr_tracking loaded in 0.51s, 106 queries (+106 other) 
2025-09-25 07:08:22,869 16948 INFO ssi_odoo_demo odoo.modules.loading: 61 modules loaded in 1.52s, 106 queries (+106 extra) 
2025-09-25 07:08:23,563 16948 INFO ssi_odoo_demo odoo.addons.base.models.ir_model: Deleting 1133@ir.ui.view (stock_qr_tracking.view_move_line_tree_inherit_qr) 
2025-09-25 07:08:23,628 16948 INFO ssi_odoo_demo odoo.models.unlink: User #1 deleted ir.model.data records with IDs: [26081] 
2025-09-25 07:08:23,628 16948 INFO ssi_odoo_demo odoo.models.unlink: User #1 deleted ir.ui.view records with IDs: [1133] 
2025-09-25 07:08:24,270 16948 INFO ssi_odoo_demo odoo.modules.loading: Modules loaded. 
2025-09-25 07:08:24,274 16948 INFO ssi_odoo_demo odoo.modules.registry: Registry loaded in 4.973s 
2025-09-25 07:08:31,881 16948 INFO ssi_odoo_demo odoo.addons.base.models.ir_http: Generating routing map for key None 
2025-09-25 07:08:32,105 16948 WARNING ssi_odoo_demo odoo.modules.module: Missing `license` key in manifest for 'stock_qr_tracking', defaulting to LGPL-3 
2025-09-25 07:08:32,212 16948 WARNING ssi_odoo_demo odoo.modules.module: Missing `license` key in manifest for 'product_master', defaulting to LGPL-3 
2025-09-25 07:08:32,398 16948 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:08:32] "GET /websocket?version=17.0-1 HTTP/1.1" 101 - 1 0.002 0.055
2025-09-25 07:08:32,584 16948 WARNING ssi_odoo_demo odoo.addons.base.models.ir_qweb: Found deprecated directive @t-esc="title or 'Odoo'" in template 180. Replace by @t-out 
2025-09-25 07:08:32,584 16948 WARNING ssi_odoo_demo odoo.addons.base.models.ir_qweb: Found deprecated directive @t-esc='request.csrf_token(None)' in template 180. Replace by @t-out 
2025-09-25 07:08:32,584 16948 WARNING ssi_odoo_demo odoo.addons.base.models.ir_qweb: Found deprecated directive @t-esc='debug' in template 180. Replace by @t-out 
2025-09-25 07:08:32,590 16948 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:08:32] "GET /web?debug=1 HTTP/1.1" 200 - 91 0.079 0.633
2025-09-25 07:08:32,751 16948 INFO ? odoo.addons.bus.models.bus: Bus.loop listen imbus on db postgres 
2025-09-25 07:08:32,905 16948 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:08:32] "GET /web/assets/99a5b6b/web.assets_web.min.css HTTP/1.1" 200 - 3 0.011 0.281
2025-09-25 07:08:33,094 16948 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:08:33] "GET /web/webclient/translations/563473cb4676ad04aeaf9e7527a74254172fcbf2?lang=en_US HTTP/1.1" 200 - 4 0.015 0.153
2025-09-25 07:08:33,102 16948 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:08:33] "GET /web/webclient/load_menus/1c55e089316d81449ff547e55c8e84de112d25bbcdb4c6c00d114e3b8e7c49f1 HTTP/1.1" 200 - 2 0.007 0.162
2025-09-25 07:08:33,638 16948 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:08:33] "GET /web/assets/b78fdf3/web.assets_web.min.js HTTP/1.1" 200 - 3 0.003 1.020
2025-09-25 07:08:34,244 16948 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:08:34] "GET /web/static/lib/odoo_ui_icons/fonts/odoo_ui_icons.woff2 HTTP/1.1" 200 - 0 0.000 0.026
2025-09-25 07:08:34,270 16948 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:08:34] "GET /web/static/src/libs/fontawesome/fonts/fontawesome-webfont.woff2?v=4.7.0 HTTP/1.1" 200 - 0 0.000 0.020
2025-09-25 07:08:34,308 16948 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:08:34] "POST /web/dataset/call_kw/res.users/systray_get_activities HTTP/1.1" 200 - 5 0.010 0.016
2025-09-25 07:08:34,373 16948 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:08:34] "GET /web/image?model=res.users&field=avatar_128&id=6 HTTP/1.1" 200 - 22 0.050 0.025
2025-09-25 07:08:34,414 16948 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:08:34] "POST /mail/init_messaging HTTP/1.1" 200 - 67 0.092 0.047
2025-09-25 07:08:34,441 16948 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:08:34] "POST /web/dataset/call_kw/stock.move.line/get_views HTTP/1.1" 200 - 40 0.080 0.077
2025-09-25 07:08:34,718 16948 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:08:34] "POST /mail/load_message_failures HTTP/1.1" 200 - 3 0.004 0.011
2025-09-25 07:08:34,720 16948 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:08:34] "GET /web/static/img/favicon.ico HTTP/1.1" 200 - 0 0.000 0.019
2025-09-25 07:08:34,776 16948 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:08:34] "POST /web/dataset/call_kw/res.users/has_group HTTP/1.1" 200 - 2 0.001 0.006
2025-09-25 07:08:34,796 16948 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:08:34] "GET /web/manifest.webmanifest HTTP/1.1" 200 - 14 0.015 0.011
2025-09-25 07:08:34,802 16948 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:08:34] "POST /web/dataset/call_kw/stock.move.line/web_search_read HTTP/1.1" 200 - 27 0.054 0.023
2025-09-25 07:08:35,131 16948 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:08:35] "GET /web/static/img/odoo-icon-192x192.png HTTP/1.1" 200 - 0 0.000 0.016
2025-09-25 07:08:36,500 16948 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:08:36] "GET /web/service-worker.js HTTP/1.1" 200 - 1 0.001 0.009
2025-09-25 07:09:24,345 16948 INFO ssi_odoo_demo odoo.addons.base.models.ir_cron: Starting job `Base: Auto-vacuum internal data`. 
2025-09-25 07:09:24,393 16948 INFO ssi_odoo_demo odoo.addons.base.models.ir_attachment: filestore gc 1 checked, 0 removed 
2025-09-25 07:09:24,449 16948 INFO ssi_odoo_demo odoo.addons.base.models.res_users: GC'd 0 user log entries 
2025-09-25 07:09:24,482 16948 INFO ssi_odoo_demo odoo.addons.auth_totp.models.auth_totp: GC'd 0 totp devices entries 
2025-09-25 07:09:24,497 16948 INFO ssi_odoo_demo odoo.models.unlink: User #1 deleted bus.bus records with IDs: [57, 58, 59] 
2025-09-25 07:09:24,582 16948 INFO ssi_odoo_demo odoo.addons.sms.models.sms_sms: GC'd 0 sms marked for deletion 
2025-09-25 07:09:24,736 16948 INFO ssi_odoo_demo odoo.addons.base.models.ir_cron: Job done: `Base: Auto-vacuum internal data` (0.391s). 
2025-09-25 07:14:35,857 16948 INFO ssi_odoo_demo odoo.service.server: Initiating shutdown 
2025-09-25 07:14:35,857 16948 INFO ssi_odoo_demo odoo.service.server: Hit CTRL-C again or send a second signal to force the shutdown. 
2025-09-25 07:14:36,409 16948 INFO ssi_odoo_demo odoo.sql_db: ConnectionPool(used=0/count=0/max=64): Closed 6 connections  
2025-09-25 07:14:39,715 7388 INFO ? odoo: Odoo version 17.0 
2025-09-25 07:14:39,715 7388 INFO ? odoo: Using configuration file at D:\work-project\ssi\odoo_demo\odoo\odoo.conf 
2025-09-25 07:14:39,715 7388 INFO ? odoo: addons paths: ['D:\\work-project\\ssi\\odoo_demo\\odoo\\odoo\\addons', 'd:\\work-project\\ssi\\odoo_demo\\odoo\\data\\addons\\17.0', 'd:\\work-project\\ssi\\odoo_demo\\odoo\\addons', 'd:\\work-project\\ssi\\odoo_demo\\odoo\\custom_addons', 'd:\\work-project\\ssi\\odoo_demo\\odoo\\odoo\\addons'] 
2025-09-25 07:14:39,715 7388 INFO ? odoo: database: odoo@localhost:5432 
2025-09-25 07:14:39,959 7388 INFO ? odoo.addons.base.models.ir_actions_report: Will use the Wkhtmltopdf binary at C:\Program Files\wkhtmltopdf\bin\wkhtmltopdf.exe 
2025-09-25 07:14:40,294 7388 WARNING ? odoo.service.server: 'watchdog' module not installed. Code autoreload feature is disabled 
2025-09-25 07:14:40,300 7388 INFO ? odoo.service.server: HTTP service (werkzeug) running on LAPTOP-O0EP5QSP:8069 
2025-09-25 07:14:40,352 7388 INFO ssi_odoo_demo odoo.modules.loading: loading 1 modules... 
2025-09-25 07:14:40,357 7388 INFO ssi_odoo_demo odoo.modules.loading: 1 modules loaded in 0.00s, 0 queries (+0 extra) 
2025-09-25 07:14:40,376 7388 INFO ssi_odoo_demo odoo.modules.loading: updating modules list 
2025-09-25 07:14:40,380 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_module: ALLOW access to module.update_list on [] to user __system__ #1 via n/a 
2025-09-25 07:14:41,546 7388 WARNING ssi_odoo_demo odoo.modules.module: Missing `license` key in manifest for 'product_master', defaulting to LGPL-3 
2025-09-25 07:14:41,725 7388 WARNING ssi_odoo_demo odoo.modules.module: Missing `license` key in manifest for 'stock_qr_tracking', defaulting to LGPL-3 
2025-09-25 07:14:42,023 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_module: ALLOW access to module.button_upgrade on ['Stock QR Tracking'] to user __system__ #1 via n/a 
2025-09-25 07:14:42,023 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_module: ALLOW access to module.update_list on ['Stock QR Tracking'] to user __system__ #1 via n/a 
2025-09-25 07:14:42,664 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_module: ALLOW access to module.button_install on [] to user __system__ #1 via n/a 
2025-09-25 07:14:42,694 7388 INFO ssi_odoo_demo odoo.modules.loading: loading 61 modules... 
2025-09-25 07:14:43,911 7388 WARNING ssi_odoo_demo odoo.api.create: The model odoo.addons.product_master.models.product_template is not overriding the create method in batch 
2025-09-25 07:14:43,957 7388 INFO ssi_odoo_demo odoo.modules.loading: Loading module stock_qr_tracking (60/61) 
2025-09-25 07:14:44,309 7388 INFO ssi_odoo_demo odoo.modules.registry: module stock_qr_tracking: creating or updating database tables 
2025-09-25 07:14:44,417 7388 INFO ssi_odoo_demo odoo.modules.loading: loading stock_qr_tracking/security/ir.model.access.csv 
2025-09-25 07:14:44,433 7388 INFO ssi_odoo_demo odoo.modules.loading: loading stock_qr_tracking/views/stock_move_line_views.xml 
2025-09-25 07:14:44,451 7388 WARNING ssi_odoo_demo odoo.addons.base.models.ir_ui_view: Could not find view arch definition in file 'd:\work-project\ssi\odoo_demo\odoo\custom_addons\stock_qr_tracking\views\stock_move_line_views.xml' for xmlid 'stock_qr_tracking.view_picking_move_line_tree_inherit_qr' 
2025-09-25 07:14:44,467 7388 WARNING ssi_odoo_demo odoo.addons.base.models.ir_ui_view: Could not find view arch definition in file 'd:\work-project\ssi\odoo_demo\odoo\custom_addons\stock_qr_tracking\views\stock_move_line_views.xml' for xmlid 'stock_qr_tracking.view_move_line_form_inherit_qr' 
2025-09-25 07:14:44,478 7388 WARNING ssi_odoo_demo odoo.modules.loading: Transient module states were reset 
2025-09-25 07:14:44,478 7388 ERROR ssi_odoo_demo odoo.modules.registry: Failed to load registry 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 575, in _tag_root
    raise ParseError(msg) from None  # Restart with "--log-handler odoo.tools.convert:DEBUG" for complete traceback
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:18
Error while parsing or validating view:

Element '<xpath expr="//group[@name=&#39;main_group&#39;]">' cannot be located in parent view

View error context:
{'file': 'd:\\work-project\\ssi\\odoo_demo\\odoo\\custom_addons\\stock_qr_tracking\\views\\stock_move_line_views.xml',
 'line': 2,
 'name': 'stock.move.line.form.qr',
 'view': ir.ui.view(1137,),
 'view.model': 'stock.move.line',
 'view.parent': ir.ui.view(867,),
 'xmlid': 'view_move_line_form_qr'}

2025-09-25 07:14:44,482 7388 CRITICAL ssi_odoo_demo odoo.service.server: Failed to initialize database `ssi_odoo_demo`. 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\service\server.py", line 1369, in preload_registries
    registry = Registry.new(dbname, update_module=update_module)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 575, in _tag_root
    raise ParseError(msg) from None  # Restart with "--log-handler odoo.tools.convert:DEBUG" for complete traceback
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:18
Error while parsing or validating view:

Element '<xpath expr="//group[@name=&#39;main_group&#39;]">' cannot be located in parent view

View error context:
{'file': 'd:\\work-project\\ssi\\odoo_demo\\odoo\\custom_addons\\stock_qr_tracking\\views\\stock_move_line_views.xml',
 'line': 2,
 'name': 'stock.move.line.form.qr',
 'view': ir.ui.view(1137,),
 'view.model': 'stock.move.line',
 'view.parent': ir.ui.view(867,),
 'xmlid': 'view_move_line_form_qr'}

2025-09-25 07:14:44,489 7388 INFO ? odoo.modules.loading: loading 1 modules... 
2025-09-25 07:14:44,496 7388 INFO ? odoo.modules.loading: 1 modules loaded in 0.01s, 0 queries (+0 extra) 
2025-09-25 07:14:44,508 7388 INFO ? odoo.modules.loading: loading 61 modules... 
2025-09-25 07:14:44,555 7388 INFO ? odoo.modules.loading: Loading module stock_qr_tracking (60/61) 
2025-09-25 07:14:44,790 7388 INFO ? odoo.modules.registry: module stock_qr_tracking: creating or updating database tables 
2025-09-25 07:14:44,841 7388 INFO ? odoo.modules.loading: loading stock_qr_tracking/security/ir.model.access.csv 
2025-09-25 07:14:44,848 7388 INFO ? odoo.modules.loading: loading stock_qr_tracking/views/stock_move_line_views.xml 
2025-09-25 07:14:44,862 7388 WARNING ? odoo.addons.base.models.ir_ui_view: Could not find view arch definition in file 'd:\work-project\ssi\odoo_demo\odoo\custom_addons\stock_qr_tracking\views\stock_move_line_views.xml' for xmlid 'stock_qr_tracking.view_picking_move_line_tree_inherit_qr' 
2025-09-25 07:14:44,879 7388 WARNING ? odoo.addons.base.models.ir_ui_view: Could not find view arch definition in file 'd:\work-project\ssi\odoo_demo\odoo\custom_addons\stock_qr_tracking\views\stock_move_line_views.xml' for xmlid 'stock_qr_tracking.view_move_line_form_inherit_qr' 
2025-09-25 07:14:44,884 7388 WARNING ? odoo.modules.loading: Transient module states were reset 
2025-09-25 07:14:44,884 7388 ERROR ? odoo.modules.registry: Failed to load registry 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 575, in _tag_root
    raise ParseError(msg) from None  # Restart with "--log-handler odoo.tools.convert:DEBUG" for complete traceback
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:18
Error while parsing or validating view:

Element '<xpath expr="//group[@name=&#39;main_group&#39;]">' cannot be located in parent view

View error context:
{'file': 'd:\\work-project\\ssi\\odoo_demo\\odoo\\custom_addons\\stock_qr_tracking\\views\\stock_move_line_views.xml',
 'line': 2,
 'name': 'stock.move.line.form.qr',
 'view': ir.ui.view(1139,),
 'view.model': 'stock.move.line',
 'view.parent': ir.ui.view(867,),
 'xmlid': 'view_move_line_form_qr'}

2025-09-25 07:14:44,886 7388 ERROR ? odoo.http: Exception during request handling. 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 2396, in __call__
    response = request._serve_db()
               ^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 1949, in _serve_db
    self.registry = Registry(self.db).check_signaling()
                    ^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 89, in __new__
    return cls.new(db_name)
           ^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 575, in _tag_root
    raise ParseError(msg) from None  # Restart with "--log-handler odoo.tools.convert:DEBUG" for complete traceback
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:18
Error while parsing or validating view:

Element '<xpath expr="//group[@name=&#39;main_group&#39;]">' cannot be located in parent view

View error context:
{'file': 'd:\\work-project\\ssi\\odoo_demo\\odoo\\custom_addons\\stock_qr_tracking\\views\\stock_move_line_views.xml',
 'line': 2,
 'name': 'stock.move.line.form.qr',
 'view': ir.ui.view(1139,),
 'view.model': 'stock.move.line',
 'view.parent': ir.ui.view(867,),
 'xmlid': 'view_move_line_form_qr'}

2025-09-25 07:14:44,894 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:14:44] "GET /websocket?version=17.0-1 HTTP/1.1" 500 - 91 0.047 4.097
2025-09-25 07:14:49,113 7388 INFO ? odoo.modules.loading: loading 1 modules... 
2025-09-25 07:14:49,118 7388 INFO ? odoo.modules.loading: 1 modules loaded in 0.00s, 0 queries (+0 extra) 
2025-09-25 07:14:49,127 7388 INFO ? odoo.modules.loading: loading 61 modules... 
2025-09-25 07:14:49,162 7388 INFO ? odoo.modules.loading: Loading module stock_qr_tracking (60/61) 
2025-09-25 07:14:49,433 7388 INFO ? odoo.modules.registry: module stock_qr_tracking: creating or updating database tables 
2025-09-25 07:14:49,485 7388 INFO ? odoo.modules.loading: loading stock_qr_tracking/security/ir.model.access.csv 
2025-09-25 07:14:49,493 7388 INFO ? odoo.modules.loading: loading stock_qr_tracking/views/stock_move_line_views.xml 
2025-09-25 07:14:49,507 7388 WARNING ? odoo.addons.base.models.ir_ui_view: Could not find view arch definition in file 'd:\work-project\ssi\odoo_demo\odoo\custom_addons\stock_qr_tracking\views\stock_move_line_views.xml' for xmlid 'stock_qr_tracking.view_picking_move_line_tree_inherit_qr' 
2025-09-25 07:14:49,520 7388 WARNING ? odoo.addons.base.models.ir_ui_view: Could not find view arch definition in file 'd:\work-project\ssi\odoo_demo\odoo\custom_addons\stock_qr_tracking\views\stock_move_line_views.xml' for xmlid 'stock_qr_tracking.view_move_line_form_inherit_qr' 
2025-09-25 07:14:49,523 7388 WARNING ? odoo.modules.loading: Transient module states were reset 
2025-09-25 07:14:49,524 7388 ERROR ? odoo.modules.registry: Failed to load registry 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 575, in _tag_root
    raise ParseError(msg) from None  # Restart with "--log-handler odoo.tools.convert:DEBUG" for complete traceback
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:18
Error while parsing or validating view:

Element '<xpath expr="//group[@name=&#39;main_group&#39;]">' cannot be located in parent view

View error context:
{'file': 'd:\\work-project\\ssi\\odoo_demo\\odoo\\custom_addons\\stock_qr_tracking\\views\\stock_move_line_views.xml',
 'line': 2,
 'name': 'stock.move.line.form.qr',
 'view': ir.ui.view(1141,),
 'view.model': 'stock.move.line',
 'view.parent': ir.ui.view(867,),
 'xmlid': 'view_move_line_form_qr'}

2025-09-25 07:14:49,526 7388 ERROR ? odoo.http: Exception during request handling. 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 2396, in __call__
    response = request._serve_db()
               ^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 1949, in _serve_db
    self.registry = Registry(self.db).check_signaling()
                    ^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 89, in __new__
    return cls.new(db_name)
           ^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 575, in _tag_root
    raise ParseError(msg) from None  # Restart with "--log-handler odoo.tools.convert:DEBUG" for complete traceback
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:18
Error while parsing or validating view:

Element '<xpath expr="//group[@name=&#39;main_group&#39;]">' cannot be located in parent view

View error context:
{'file': 'd:\\work-project\\ssi\\odoo_demo\\odoo\\custom_addons\\stock_qr_tracking\\views\\stock_move_line_views.xml',
 'line': 2,
 'name': 'stock.move.line.form.qr',
 'view': ir.ui.view(1141,),
 'view.model': 'stock.move.line',
 'view.parent': ir.ui.view(867,),
 'xmlid': 'view_move_line_form_qr'}

2025-09-25 07:14:49,527 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:14:49] "GET /websocket?version=17.0-1 HTTP/1.1" 500 - 91 0.045 0.374
2025-09-25 07:14:56,646 7388 INFO ? odoo.modules.loading: loading 1 modules... 
2025-09-25 07:14:56,650 7388 INFO ? odoo.modules.loading: 1 modules loaded in 0.00s, 0 queries (+0 extra) 
2025-09-25 07:14:56,657 7388 INFO ? odoo.modules.loading: loading 61 modules... 
2025-09-25 07:14:56,693 7388 INFO ? odoo.modules.loading: Loading module stock_qr_tracking (60/61) 
2025-09-25 07:14:56,900 7388 INFO ? odoo.modules.registry: module stock_qr_tracking: creating or updating database tables 
2025-09-25 07:14:56,953 7388 INFO ? odoo.modules.loading: loading stock_qr_tracking/security/ir.model.access.csv 
2025-09-25 07:14:56,960 7388 INFO ? odoo.modules.loading: loading stock_qr_tracking/views/stock_move_line_views.xml 
2025-09-25 07:14:56,975 7388 WARNING ? odoo.addons.base.models.ir_ui_view: Could not find view arch definition in file 'd:\work-project\ssi\odoo_demo\odoo\custom_addons\stock_qr_tracking\views\stock_move_line_views.xml' for xmlid 'stock_qr_tracking.view_picking_move_line_tree_inherit_qr' 
2025-09-25 07:14:56,989 7388 WARNING ? odoo.addons.base.models.ir_ui_view: Could not find view arch definition in file 'd:\work-project\ssi\odoo_demo\odoo\custom_addons\stock_qr_tracking\views\stock_move_line_views.xml' for xmlid 'stock_qr_tracking.view_move_line_form_inherit_qr' 
2025-09-25 07:14:56,992 7388 WARNING ? odoo.modules.loading: Transient module states were reset 
2025-09-25 07:14:56,992 7388 ERROR ? odoo.modules.registry: Failed to load registry 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 575, in _tag_root
    raise ParseError(msg) from None  # Restart with "--log-handler odoo.tools.convert:DEBUG" for complete traceback
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:18
Error while parsing or validating view:

Element '<xpath expr="//group[@name=&#39;main_group&#39;]">' cannot be located in parent view

View error context:
{'file': 'd:\\work-project\\ssi\\odoo_demo\\odoo\\custom_addons\\stock_qr_tracking\\views\\stock_move_line_views.xml',
 'line': 2,
 'name': 'stock.move.line.form.qr',
 'view': ir.ui.view(1143,),
 'view.model': 'stock.move.line',
 'view.parent': ir.ui.view(867,),
 'xmlid': 'view_move_line_form_qr'}

2025-09-25 07:14:56,994 7388 ERROR ? odoo.http: Exception during request handling. 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 2396, in __call__
    response = request._serve_db()
               ^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 1949, in _serve_db
    self.registry = Registry(self.db).check_signaling()
                    ^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 89, in __new__
    return cls.new(db_name)
           ^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 575, in _tag_root
    raise ParseError(msg) from None  # Restart with "--log-handler odoo.tools.convert:DEBUG" for complete traceback
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:18
Error while parsing or validating view:

Element '<xpath expr="//group[@name=&#39;main_group&#39;]">' cannot be located in parent view

View error context:
{'file': 'd:\\work-project\\ssi\\odoo_demo\\odoo\\custom_addons\\stock_qr_tracking\\views\\stock_move_line_views.xml',
 'line': 2,
 'name': 'stock.move.line.form.qr',
 'view': ir.ui.view(1143,),
 'view.model': 'stock.move.line',
 'view.parent': ir.ui.view(867,),
 'xmlid': 'view_move_line_form_qr'}

2025-09-25 07:14:56,995 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:14:56] "GET /websocket?version=17.0-1 HTTP/1.1" 500 - 91 0.041 0.314
2025-09-25 07:15:08,030 7388 INFO ? odoo.modules.loading: loading 1 modules... 
2025-09-25 07:15:08,035 7388 INFO ? odoo.modules.loading: 1 modules loaded in 0.01s, 0 queries (+0 extra) 
2025-09-25 07:15:08,043 7388 INFO ? odoo.modules.loading: loading 61 modules... 
2025-09-25 07:15:08,081 7388 INFO ? odoo.modules.loading: Loading module stock_qr_tracking (60/61) 
2025-09-25 07:15:08,367 7388 INFO ? odoo.modules.registry: module stock_qr_tracking: creating or updating database tables 
2025-09-25 07:15:08,420 7388 INFO ? odoo.modules.loading: loading stock_qr_tracking/security/ir.model.access.csv 
2025-09-25 07:15:08,427 7388 INFO ? odoo.modules.loading: loading stock_qr_tracking/views/stock_move_line_views.xml 
2025-09-25 07:15:08,441 7388 WARNING ? odoo.addons.base.models.ir_ui_view: Could not find view arch definition in file 'd:\work-project\ssi\odoo_demo\odoo\custom_addons\stock_qr_tracking\views\stock_move_line_views.xml' for xmlid 'stock_qr_tracking.view_picking_move_line_tree_inherit_qr' 
2025-09-25 07:15:08,455 7388 WARNING ? odoo.addons.base.models.ir_ui_view: Could not find view arch definition in file 'd:\work-project\ssi\odoo_demo\odoo\custom_addons\stock_qr_tracking\views\stock_move_line_views.xml' for xmlid 'stock_qr_tracking.view_move_line_form_inherit_qr' 
2025-09-25 07:15:08,459 7388 WARNING ? odoo.modules.loading: Transient module states were reset 
2025-09-25 07:15:08,459 7388 ERROR ? odoo.modules.registry: Failed to load registry 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 575, in _tag_root
    raise ParseError(msg) from None  # Restart with "--log-handler odoo.tools.convert:DEBUG" for complete traceback
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:18
Error while parsing or validating view:

Element '<xpath expr="//group[@name=&#39;main_group&#39;]">' cannot be located in parent view

View error context:
{'file': 'd:\\work-project\\ssi\\odoo_demo\\odoo\\custom_addons\\stock_qr_tracking\\views\\stock_move_line_views.xml',
 'line': 2,
 'name': 'stock.move.line.form.qr',
 'view': ir.ui.view(1145,),
 'view.model': 'stock.move.line',
 'view.parent': ir.ui.view(867,),
 'xmlid': 'view_move_line_form_qr'}

2025-09-25 07:15:08,460 7388 ERROR ? odoo.http: Exception during request handling. 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 2396, in __call__
    response = request._serve_db()
               ^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 1949, in _serve_db
    self.registry = Registry(self.db).check_signaling()
                    ^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 89, in __new__
    return cls.new(db_name)
           ^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 575, in _tag_root
    raise ParseError(msg) from None  # Restart with "--log-handler odoo.tools.convert:DEBUG" for complete traceback
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:18
Error while parsing or validating view:

Element '<xpath expr="//group[@name=&#39;main_group&#39;]">' cannot be located in parent view

View error context:
{'file': 'd:\\work-project\\ssi\\odoo_demo\\odoo\\custom_addons\\stock_qr_tracking\\views\\stock_move_line_views.xml',
 'line': 2,
 'name': 'stock.move.line.form.qr',
 'view': ir.ui.view(1145,),
 'view.model': 'stock.move.line',
 'view.parent': ir.ui.view(867,),
 'xmlid': 'view_move_line_form_qr'}

2025-09-25 07:15:08,461 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:08] "GET /websocket?version=17.0-1 HTTP/1.1" 500 - 91 0.040 0.398
2025-09-25 07:15:20,322 7388 INFO ? odoo.modules.loading: loading 1 modules... 
2025-09-25 07:15:20,328 7388 INFO ? odoo.modules.loading: 1 modules loaded in 0.01s, 0 queries (+0 extra) 
2025-09-25 07:15:20,336 7388 INFO ? odoo.modules.loading: loading 61 modules... 
2025-09-25 07:15:20,377 7388 INFO ? odoo.modules.loading: Loading module stock_qr_tracking (60/61) 
2025-09-25 07:15:20,592 7388 INFO ? odoo.modules.registry: module stock_qr_tracking: creating or updating database tables 
2025-09-25 07:15:20,646 7388 INFO ? odoo.modules.loading: loading stock_qr_tracking/security/ir.model.access.csv 
2025-09-25 07:15:20,652 7388 INFO ? odoo.modules.loading: loading stock_qr_tracking/views/stock_move_line_views.xml 
2025-09-25 07:15:20,666 7388 WARNING ? odoo.addons.base.models.ir_ui_view: Could not find view arch definition in file 'd:\work-project\ssi\odoo_demo\odoo\custom_addons\stock_qr_tracking\views\stock_move_line_views.xml' for xmlid 'stock_qr_tracking.view_picking_move_line_tree_inherit_qr' 
2025-09-25 07:15:20,680 7388 WARNING ? odoo.addons.base.models.ir_ui_view: Could not find view arch definition in file 'd:\work-project\ssi\odoo_demo\odoo\custom_addons\stock_qr_tracking\views\stock_move_line_views.xml' for xmlid 'stock_qr_tracking.view_move_line_form_inherit_qr' 
2025-09-25 07:15:20,684 7388 WARNING ? odoo.modules.loading: Transient module states were reset 
2025-09-25 07:15:20,684 7388 ERROR ? odoo.modules.registry: Failed to load registry 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 575, in _tag_root
    raise ParseError(msg) from None  # Restart with "--log-handler odoo.tools.convert:DEBUG" for complete traceback
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:18
Error while parsing or validating view:

Element '<xpath expr="//group[@name=&#39;main_group&#39;]">' cannot be located in parent view

View error context:
{'file': 'd:\\work-project\\ssi\\odoo_demo\\odoo\\custom_addons\\stock_qr_tracking\\views\\stock_move_line_views.xml',
 'line': 2,
 'name': 'stock.move.line.form.qr',
 'view': ir.ui.view(1147,),
 'view.model': 'stock.move.line',
 'view.parent': ir.ui.view(867,),
 'xmlid': 'view_move_line_form_qr'}

2025-09-25 07:15:20,685 7388 ERROR ? odoo.http: Exception during request handling. 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 2396, in __call__
    response = request._serve_db()
               ^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 1949, in _serve_db
    self.registry = Registry(self.db).check_signaling()
                    ^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 89, in __new__
    return cls.new(db_name)
           ^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 575, in _tag_root
    raise ParseError(msg) from None  # Restart with "--log-handler odoo.tools.convert:DEBUG" for complete traceback
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:18
Error while parsing or validating view:

Element '<xpath expr="//group[@name=&#39;main_group&#39;]">' cannot be located in parent view

View error context:
{'file': 'd:\\work-project\\ssi\\odoo_demo\\odoo\\custom_addons\\stock_qr_tracking\\views\\stock_move_line_views.xml',
 'line': 2,
 'name': 'stock.move.line.form.qr',
 'view': ir.ui.view(1147,),
 'view.model': 'stock.move.line',
 'view.parent': ir.ui.view(867,),
 'xmlid': 'view_move_line_form_qr'}

2025-09-25 07:15:20,686 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:20] "GET /web?debug=1 HTTP/1.1" 500 - 91 0.044 0.326
2025-09-25 07:15:22,713 7388 INFO ? odoo.modules.loading: loading 1 modules... 
2025-09-25 07:15:22,724 7388 INFO ? odoo.modules.loading: 1 modules loaded in 0.01s, 0 queries (+0 extra) 
2025-09-25 07:15:22,742 7388 INFO ? odoo.modules.loading: loading 61 modules... 
2025-09-25 07:15:22,781 7388 INFO ? odoo.modules.loading: Loading module stock_qr_tracking (60/61) 
2025-09-25 07:15:23,200 7388 INFO ? odoo.modules.registry: module stock_qr_tracking: creating or updating database tables 
2025-09-25 07:15:23,258 7388 INFO ? odoo.modules.loading: loading stock_qr_tracking/security/ir.model.access.csv 
2025-09-25 07:15:23,262 7388 INFO ? odoo.modules.loading: loading stock_qr_tracking/views/stock_move_line_views.xml 
2025-09-25 07:15:23,277 7388 WARNING ? odoo.addons.base.models.ir_ui_view: Could not find view arch definition in file 'd:\work-project\ssi\odoo_demo\odoo\custom_addons\stock_qr_tracking\views\stock_move_line_views.xml' for xmlid 'stock_qr_tracking.view_picking_move_line_tree_inherit_qr' 
2025-09-25 07:15:23,293 7388 WARNING ? odoo.addons.base.models.ir_ui_view: Could not find view arch definition in file 'd:\work-project\ssi\odoo_demo\odoo\custom_addons\stock_qr_tracking\views\stock_move_line_views.xml' for xmlid 'stock_qr_tracking.view_move_line_form_inherit_qr' 
2025-09-25 07:15:23,296 7388 WARNING ? odoo.modules.loading: Transient module states were reset 
2025-09-25 07:15:23,296 7388 ERROR ? odoo.modules.registry: Failed to load registry 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 575, in _tag_root
    raise ParseError(msg) from None  # Restart with "--log-handler odoo.tools.convert:DEBUG" for complete traceback
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:18
Error while parsing or validating view:

Element '<xpath expr="//group[@name=&#39;main_group&#39;]">' cannot be located in parent view

View error context:
{'file': 'd:\\work-project\\ssi\\odoo_demo\\odoo\\custom_addons\\stock_qr_tracking\\views\\stock_move_line_views.xml',
 'line': 2,
 'name': 'stock.move.line.form.qr',
 'view': ir.ui.view(1149,),
 'view.model': 'stock.move.line',
 'view.parent': ir.ui.view(867,),
 'xmlid': 'view_move_line_form_qr'}

2025-09-25 07:15:23,299 7388 ERROR ? odoo.http: Exception during request handling. 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 2396, in __call__
    response = request._serve_db()
               ^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 1949, in _serve_db
    self.registry = Registry(self.db).check_signaling()
                    ^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 89, in __new__
    return cls.new(db_name)
           ^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 575, in _tag_root
    raise ParseError(msg) from None  # Restart with "--log-handler odoo.tools.convert:DEBUG" for complete traceback
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:18
Error while parsing or validating view:

Element '<xpath expr="//group[@name=&#39;main_group&#39;]">' cannot be located in parent view

View error context:
{'file': 'd:\\work-project\\ssi\\odoo_demo\\odoo\\custom_addons\\stock_qr_tracking\\views\\stock_move_line_views.xml',
 'line': 2,
 'name': 'stock.move.line.form.qr',
 'view': ir.ui.view(1149,),
 'view.model': 'stock.move.line',
 'view.parent': ir.ui.view(867,),
 'xmlid': 'view_move_line_form_qr'}

2025-09-25 07:15:23,301 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:23] "GET /web/service-worker.js HTTP/1.1" 500 - 91 0.046 0.549
2025-09-25 07:15:25,347 7388 INFO ? odoo.modules.loading: loading 1 modules... 
2025-09-25 07:15:25,353 7388 INFO ? odoo.modules.loading: 1 modules loaded in 0.01s, 0 queries (+0 extra) 
2025-09-25 07:15:25,361 7388 INFO ? odoo.modules.loading: loading 61 modules... 
2025-09-25 07:15:25,402 7388 INFO ? odoo.modules.loading: Loading module stock_qr_tracking (60/61) 
2025-09-25 07:15:25,621 7388 INFO ? odoo.modules.registry: module stock_qr_tracking: creating or updating database tables 
2025-09-25 07:15:25,674 7388 INFO ? odoo.modules.loading: loading stock_qr_tracking/security/ir.model.access.csv 
2025-09-25 07:15:25,681 7388 INFO ? odoo.modules.loading: loading stock_qr_tracking/views/stock_move_line_views.xml 
2025-09-25 07:15:25,696 7388 WARNING ? odoo.addons.base.models.ir_ui_view: Could not find view arch definition in file 'd:\work-project\ssi\odoo_demo\odoo\custom_addons\stock_qr_tracking\views\stock_move_line_views.xml' for xmlid 'stock_qr_tracking.view_picking_move_line_tree_inherit_qr' 
2025-09-25 07:15:25,708 7388 WARNING ? odoo.addons.base.models.ir_ui_view: Could not find view arch definition in file 'd:\work-project\ssi\odoo_demo\odoo\custom_addons\stock_qr_tracking\views\stock_move_line_views.xml' for xmlid 'stock_qr_tracking.view_move_line_form_inherit_qr' 
2025-09-25 07:15:25,712 7388 WARNING ? odoo.modules.loading: Transient module states were reset 
2025-09-25 07:15:25,712 7388 ERROR ? odoo.modules.registry: Failed to load registry 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 575, in _tag_root
    raise ParseError(msg) from None  # Restart with "--log-handler odoo.tools.convert:DEBUG" for complete traceback
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:18
Error while parsing or validating view:

Element '<xpath expr="//group[@name=&#39;main_group&#39;]">' cannot be located in parent view

View error context:
{'file': 'd:\\work-project\\ssi\\odoo_demo\\odoo\\custom_addons\\stock_qr_tracking\\views\\stock_move_line_views.xml',
 'line': 2,
 'name': 'stock.move.line.form.qr',
 'view': ir.ui.view(1151,),
 'view.model': 'stock.move.line',
 'view.parent': ir.ui.view(867,),
 'xmlid': 'view_move_line_form_qr'}

2025-09-25 07:15:25,713 7388 ERROR ? odoo.http: Exception during request handling. 
Traceback (most recent call last):
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 2396, in __call__
    response = request._serve_db()
               ^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\http.py", line 1949, in _serve_db
    self.registry = Registry(self.db).check_signaling()
                    ^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 89, in __new__
    return cls.new(db_name)
           ^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\venv\Lib\site-packages\decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\func.py", line 87, in locked
    return func(inst, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\registry.py", line 110, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 481, in load_modules
    processed_modules += load_marked_modules(env, graph,
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 366, in load_marked_modules
    loaded, processed = load_module_graph(
                        ^^^^^^^^^^^^^^^^^^
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 229, in load_module_graph
    load_data(env, idref, mode, kind='data', package=package)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\modules\loading.py", line 73, in load_data
    tools.convert_file(env, package.name, filename, idref, mode, noupdate, kind)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 635, in convert_file
    convert_xml_import(env, module, fp, idref, mode, noupdate)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 701, in convert_xml_import
    obj.parse(doc.getroot())
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 621, in parse
    self._tag_root(de)
  File "D:\work-project\ssi\odoo_demo\odoo\odoo\tools\convert.py", line 575, in _tag_root
    raise ParseError(msg) from None  # Restart with "--log-handler odoo.tools.convert:DEBUG" for complete traceback
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
odoo.tools.convert.ParseError: while parsing file:/d:/work-project/ssi/odoo_demo/odoo/custom_addons/stock_qr_tracking/views/stock_move_line_views.xml:18
Error while parsing or validating view:

Element '<xpath expr="//group[@name=&#39;main_group&#39;]">' cannot be located in parent view

View error context:
{'file': 'd:\\work-project\\ssi\\odoo_demo\\odoo\\custom_addons\\stock_qr_tracking\\views\\stock_move_line_views.xml',
 'line': 2,
 'name': 'stock.move.line.form.qr',
 'view': ir.ui.view(1151,),
 'view.model': 'stock.move.line',
 'view.parent': ir.ui.view(867,),
 'xmlid': 'view_move_line_form_qr'}

2025-09-25 07:15:25,714 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:25] "GET /websocket?version=17.0-1 HTTP/1.1" 500 - 91 0.047 0.327
2025-09-25 07:15:29,781 7388 INFO ? odoo.modules.loading: loading 1 modules... 
2025-09-25 07:15:29,786 7388 INFO ? odoo.modules.loading: 1 modules loaded in 0.00s, 0 queries (+0 extra) 
2025-09-25 07:15:29,794 7388 INFO ? odoo.modules.loading: loading 61 modules... 
2025-09-25 07:15:29,832 7388 INFO ? odoo.modules.loading: Loading module stock_qr_tracking (60/61) 
2025-09-25 07:15:30,152 7388 INFO ? odoo.modules.registry: module stock_qr_tracking: creating or updating database tables 
2025-09-25 07:15:30,212 7388 INFO ? odoo.modules.loading: loading stock_qr_tracking/security/ir.model.access.csv 
2025-09-25 07:15:30,218 7388 INFO ? odoo.modules.loading: loading stock_qr_tracking/views/stock_move_line_views.xml 
2025-09-25 07:15:30,277 7388 INFO ? odoo.modules.loading: Module stock_qr_tracking loaded in 0.44s, 105 queries (+105 other) 
2025-09-25 07:15:30,293 7388 INFO ? odoo.modules.loading: 61 modules loaded in 0.50s, 105 queries (+105 extra) 
2025-09-25 07:15:30,613 7388 INFO ? odoo.modules.loading: Modules loaded. 
2025-09-25 07:15:30,616 7388 INFO ? odoo.modules.registry: Registry loaded in 0.839s 
2025-09-25 07:15:30,617 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_http: Generating routing map for key None 
2025-09-25 07:15:30,842 7388 WARNING ssi_odoo_demo odoo.modules.module: Missing `license` key in manifest for 'product_master', defaulting to LGPL-3 
2025-09-25 07:15:30,864 7388 WARNING ssi_odoo_demo odoo.modules.module: Missing `license` key in manifest for 'stock_qr_tracking', defaulting to LGPL-3 
2025-09-25 07:15:31,198 7388 WARNING ssi_odoo_demo odoo.addons.base.models.ir_qweb: Found deprecated directive @t-esc="title or 'Odoo'" in template 180. Replace by @t-out 
2025-09-25 07:15:31,198 7388 WARNING ssi_odoo_demo odoo.addons.base.models.ir_qweb: Found deprecated directive @t-esc='request.csrf_token(None)' in template 180. Replace by @t-out 
2025-09-25 07:15:31,199 7388 WARNING ssi_odoo_demo odoo.addons.base.models.ir_qweb: Found deprecated directive @t-esc='debug' in template 180. Replace by @t-out 
2025-09-25 07:15:31,203 7388 INFO ssi_odoo_demo odoo.modules.registry: Caches invalidated, signaling through the database: ['default', 'templates'] 
2025-09-25 07:15:31,253 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:31] "GET /web?debug=1 HTTP/1.1" 200 - 533 0.329 1.148
2025-09-25 07:15:31,641 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:31] "POST /mail/init_messaging HTTP/1.1" 200 - 69 0.074 0.054
2025-09-25 07:15:31,701 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:31] "POST /web/dataset/call_kw/res.users/systray_get_activities HTTP/1.1" 200 - 6 0.006 0.130
2025-09-25 07:15:31,820 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:31] "GET /websocket?version=17.0-1 HTTP/1.1" 101 - 1 0.001 0.008
2025-09-25 07:15:31,849 7388 INFO ? odoo.addons.bus.models.bus: Bus.loop listen imbus on db postgres 
2025-09-25 07:15:31,966 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:31] "POST /mail/load_message_failures HTTP/1.1" 200 - 3 0.019 0.004
2025-09-25 07:15:31,982 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:31] "POST /web/dataset/call_kw/stock.move.line/get_views HTTP/1.1" 200 - 41 0.030 0.043
2025-09-25 07:15:32,021 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:32] "GET /web/image?model=res.users&field=avatar_128&id=6 HTTP/1.1" 304 - 22 0.042 0.015
2025-09-25 07:15:32,315 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:32] "POST /web/dataset/call_kw/res.users/has_group HTTP/1.1" 200 - 2 0.004 0.001
2025-09-25 07:15:32,428 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:32] "GET /web/manifest.webmanifest HTTP/1.1" 200 - 14 0.076 0.010
2025-09-25 07:15:32,444 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:32] "POST /web/dataset/call_kw/stock.move.line/web_search_read HTTP/1.1" 200 - 27 0.050 0.087
2025-09-25 07:15:34,349 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:34] "GET /web/service-worker.js HTTP/1.1" 200 - 1 0.000 0.005
2025-09-25 07:15:51,797 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:51] "POST /web/action/load HTTP/1.1" 200 - 9 0.013 0.008
2025-09-25 07:15:52,126 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:52] "POST /web/dataset/call_kw/ir.module.module/get_views HTTP/1.1" 200 - 50 0.030 0.057
2025-09-25 07:15:52,136 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:52] "POST /web/dataset/call_kw/ir.module.module/search_panel_select_range HTTP/1.1" 200 - 1 0.001 0.003
2025-09-25 07:15:52,503 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:52] "POST /web/dataset/call_kw/ir.module.module/search_panel_select_range HTTP/1.1" 200 - 65 0.036 0.021
2025-09-25 07:15:52,550 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:52] "POST /web/dataset/call_kw/ir.module.module/web_search_read HTTP/1.1" 200 - 8 0.005 0.099
2025-09-25 07:15:53,102 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:53] "GET /base/static/img/icons/mrp_workorder.png HTTP/1.1" 200 - 0 0.000 0.005
2025-09-25 07:15:53,104 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:53] "GET /crm/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.006
2025-09-25 07:15:53,104 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:53] "GET /account/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.005
2025-09-25 07:15:53,106 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:53] "GET /pos_restaurant/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.009
2025-09-25 07:15:53,107 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:53] "GET /sale_management/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.010
2025-09-25 07:15:53,109 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:53] "GET /website/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.010
2025-09-25 07:15:53,418 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:53] "GET /stock/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.006
2025-09-25 07:15:53,430 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:53] "GET /base/static/img/icons/account_accountant.png HTTP/1.1" 200 - 0 0.000 0.007
2025-09-25 07:15:53,432 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:53] "GET /point_of_sale/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.007
2025-09-25 07:15:53,433 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:53] "GET /base/static/img/icons/knowledge.png HTTP/1.1" 200 - 0 0.000 0.008
2025-09-25 07:15:53,434 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:53] "GET /purchase/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.008
2025-09-25 07:15:53,434 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:53] "GET /project/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.008
2025-09-25 07:15:53,737 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:53] "GET /website_sale/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.003
2025-09-25 07:15:53,738 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:53] "GET /mrp/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.003
2025-09-25 07:15:53,744 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:53] "GET /mass_mailing/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.004
2025-09-25 07:15:53,749 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:53] "GET /base/static/img/icons/timesheet_grid.png HTTP/1.1" 200 - 0 0.000 0.008
2025-09-25 07:15:53,753 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:53] "GET /base/static/img/icons/web_studio.png HTTP/1.1" 200 - 0 0.000 0.004
2025-09-25 07:15:53,753 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:53] "GET /hr_expense/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.006
2025-09-25 07:15:54,075 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:54] "GET /hr_holidays/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.012
2025-09-25 07:15:54,081 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:54] "GET /hr_recruitment/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.017
2025-09-25 07:15:54,087 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:54] "GET /hr/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.018
2025-09-25 07:15:54,087 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:54] "GET /base/static/img/icons/industry_fsm.png HTTP/1.1" 200 - 0 0.000 0.020
2025-09-25 07:15:54,120 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:54] "POST /web/dataset/call_kw/ir.module.module/search_panel_select_range HTTP/1.1" 200 - 64 0.050 0.026
2025-09-25 07:15:54,176 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:54] "POST /web/dataset/call_kw/ir.module.module/web_search_read HTTP/1.1" 200 - 11 0.016 0.098
2025-09-25 07:15:54,389 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:54] "GET /data_recycle/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.003
2025-09-25 07:15:54,433 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:54] "GET /base/static/img/icons/sign.png HTTP/1.1" 200 - 0 0.000 0.002
2025-09-25 07:15:54,437 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:54] "GET /maintenance/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.007
2025-09-25 07:15:54,448 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:54] "GET /base/static/img/icons/helpdesk.png HTTP/1.1" 200 - 0 0.000 0.006
2025-09-25 07:15:54,450 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:54] "GET /base/static/img/icons/sale_subscription.png HTTP/1.1" 200 - 0 0.000 0.004
2025-09-25 07:15:54,484 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:54] "GET /base/static/img/icons/quality_control.png HTTP/1.1" 200 - 0 0.000 0.002
2025-09-25 07:15:54,701 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:54] "GET /website_slides/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.005
2025-09-25 07:15:54,770 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:54] "GET /base/static/img/icons/planning.png HTTP/1.1" 200 - 0 0.000 0.009
2025-09-25 07:15:54,773 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:54] "GET /website_event/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.012
2025-09-25 07:15:54,775 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:54] "GET /mail/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.012
2025-09-25 07:15:54,777 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:54] "GET /contacts/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.013
2025-09-25 07:15:54,791 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:54] "GET /base/static/img/icons/mrp_plm.png HTTP/1.1" 200 - 0 0.000 0.003
2025-09-25 07:15:55,011 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:55] "GET /calendar/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.004
2025-09-25 07:15:55,084 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:55] "GET /base/static/img/icons/social.png HTTP/1.1" 200 - 0 0.000 0.003
2025-09-25 07:15:55,096 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:55] "GET /base/static/img/icons/hr_appraisal.png HTTP/1.1" 200 - 0 0.000 0.006
2025-09-25 07:15:55,099 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:55] "GET /fleet/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.008
2025-09-25 07:15:55,100 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:55] "GET /base/static/img/icons/marketing_automation.png HTTP/1.1" 200 - 0 0.000 0.005
2025-09-25 07:15:55,104 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:55] "GET /im_livechat/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.004
2025-09-25 07:15:55,326 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:55] "GET /base/static/img/icons/appointment.png HTTP/1.1" 200 - 0 0.000 0.004
2025-09-25 07:15:55,400 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:55] "GET /survey/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.002
2025-09-25 07:15:55,405 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:55] "GET /base/static/img/icons/web_mobile.png HTTP/1.1" 200 - 0 0.000 0.002
2025-09-25 07:15:55,412 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:55] "GET /repair/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.004
2025-09-25 07:15:55,413 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:55] "GET /hr_attendance/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.005
2025-09-25 07:15:55,415 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:55] "GET /mass_mailing_sms/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.005
2025-09-25 07:15:55,637 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:55] "GET /base/static/img/icons/stock_barcode.png HTTP/1.1" 200 - 0 0.000 0.005
2025-09-25 07:15:55,714 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:55] "GET /project_todo/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.004
2025-09-25 07:15:55,721 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:55] "GET /hr_skills/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.005
2025-09-25 07:15:55,722 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:55] "GET /base/static/img/icons/voip.png HTTP/1.1" 200 - 0 0.000 0.004
2025-09-25 07:15:55,727 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:55] "GET /lunch/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.005
2025-09-25 07:15:55,728 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:55] "GET /website_hr_recruitment/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.004
2025-09-25 07:15:55,946 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:55] "GET /base/static/img/icons/sale_amazon.png HTTP/1.1" 200 - 0 0.000 0.002
2025-09-25 07:15:56,024 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:56] "GET /base/static/img/icons/sale_ebay.png HTTP/1.1" 200 - 0 0.000 0.001
2025-09-25 07:15:56,031 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:56] "GET /hr_contract/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.003
2025-09-25 07:15:56,033 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:56] "GET /base/static/description/icon.png HTTP/1.1" 200 - 0 0.000 0.003
2025-09-25 07:15:57,778 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:57] "POST /web/dataset/call_kw/ir.module.module/search_panel_select_range HTTP/1.1" 200 - 64 0.058 0.020
2025-09-25 07:15:57,980 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:15:57] "POST /web/dataset/call_kw/ir.module.module/web_search_read HTTP/1.1" 200 - 8 0.007 0.010
2025-09-25 07:16:02,290 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_module: ALLOW access to module.button_uninstall_wizard on ['Stock QR Tracking'] to user faqihazh2@gmail.com #6 via 127.0.0.1 
2025-09-25 07:16:02,291 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:02] "POST /web/dataset/call_button HTTP/1.1" 200 - 3 0.003 0.004
2025-09-25 07:16:02,672 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:02] "POST /web/dataset/call_kw/base.module.uninstall/get_views HTTP/1.1" 200 - 10 0.007 0.050
2025-09-25 07:16:02,887 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:02] "POST /web/dataset/call_kw/base.module.uninstall/onchange HTTP/1.1" 200 - 15 0.018 0.010
2025-09-25 07:16:04,657 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:04] "POST /web/dataset/call_kw/base.module.uninstall/web_save HTTP/1.1" 200 - 18 0.011 0.016
2025-09-25 07:16:04,980 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_module: ALLOW access to module.button_immediate_uninstall on ['Stock QR Tracking'] to user faqihazh2@gmail.com #6 via 127.0.0.1 
2025-09-25 07:16:04,980 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_module: User #6 triggered module uninstallation 
2025-09-25 07:16:04,980 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_module: ALLOW access to module.button_uninstall on ['Stock QR Tracking'] to user faqihazh2@gmail.com #6 via 127.0.0.1 
2025-09-25 07:16:05,020 7388 INFO ssi_odoo_demo odoo.modules.loading: loading 1 modules... 
2025-09-25 07:16:05,028 7388 INFO ssi_odoo_demo odoo.modules.loading: 1 modules loaded in 0.01s, 0 queries (+0 extra) 
2025-09-25 07:16:05,054 7388 INFO ssi_odoo_demo odoo.modules.loading: updating modules list 
2025-09-25 07:16:05,056 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_module: ALLOW access to module.update_list on [] to user __system__ #1 via 127.0.0.1 
2025-09-25 07:16:05,698 7388 INFO ssi_odoo_demo odoo.modules.loading: loading 61 modules... 
2025-09-25 07:16:05,741 7388 INFO ssi_odoo_demo odoo.modules.loading: 61 modules loaded in 0.04s, 0 queries (+0 extra) 
2025-09-25 07:16:06,009 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_module: ALLOW access to module.module_uninstall on ['Stock QR Tracking'] to user __system__ #1 via 127.0.0.1 
2025-09-25 07:16:06,013 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_model: Deleting ir.ui.view(1135, 1134) 
2025-09-25 07:16:06,066 7388 INFO ssi_odoo_demo odoo.models.unlink: User #1 deleted ir.model.data records with IDs: [26082, 26743] 
2025-09-25 07:16:06,066 7388 INFO ssi_odoo_demo odoo.models.unlink: User #1 deleted ir.ui.view records with IDs: [1135, 1134] 
2025-09-25 07:16:06,067 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_model: Deleting ir.model.access(626,) 
2025-09-25 07:16:06,078 7388 INFO ssi_odoo_demo odoo.models.unlink: User #1 deleted ir.model.data records with IDs: [26080] 
2025-09-25 07:16:06,078 7388 INFO ssi_odoo_demo odoo.models.unlink: User #1 deleted ir.model.access records with IDs: [626] 
2025-09-25 07:16:06,090 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_model: Deleting ir.model.fields(7150, 7149, 7148) 
2025-09-25 07:16:06,157 7388 INFO ssi_odoo_demo odoo.models.unlink: User #1 deleted ir.model.data records with IDs: [26055, 26053, 26054] 
2025-09-25 07:16:06,157 7388 INFO ssi_odoo_demo odoo.models.unlink: User #1 deleted ir.model.fields records with IDs: [7150, 7149, 7148] 
2025-09-25 07:16:06,162 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_model: ir.model.data could not be deleted ([]) 
2025-09-25 07:16:06,165 7388 INFO ssi_odoo_demo odoo.models.unlink: User #1 deleted ir.model.data records with IDs: [26743, 26082, 26080, 26055, 26054, 26053, 26052, 26051] 
2025-09-25 07:16:06,174 7388 INFO ssi_odoo_demo odoo.modules.loading: Reloading registry once more after uninstalling modules 
2025-09-25 07:16:06,234 7388 INFO ssi_odoo_demo odoo.modules.loading: loading 1 modules... 
2025-09-25 07:16:06,238 7388 INFO ssi_odoo_demo odoo.modules.loading: 1 modules loaded in 0.00s, 0 queries (+0 extra) 
2025-09-25 07:16:06,260 7388 INFO ssi_odoo_demo odoo.modules.loading: updating modules list 
2025-09-25 07:16:06,265 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_module: ALLOW access to module.update_list on [] to user __system__ #1 via 127.0.0.1 
2025-09-25 07:16:06,970 7388 INFO ssi_odoo_demo odoo.modules.loading: loading 60 modules... 
2025-09-25 07:16:07,009 7388 INFO ssi_odoo_demo odoo.modules.loading: 60 modules loaded in 0.04s, 0 queries (+0 extra) 
2025-09-25 07:16:08,013 7388 INFO ssi_odoo_demo odoo.modules.loading: Modules loaded. 
2025-09-25 07:16:08,017 7388 INFO ssi_odoo_demo odoo.modules.registry: Registry loaded in 1.843s 
2025-09-25 07:16:08,021 7388 INFO ssi_odoo_demo odoo.modules.registry: Registry loaded in 3.023s 
2025-09-25 07:16:08,021 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_module: getting next ir.actions.todo() 
2025-09-25 07:16:08,027 7388 INFO ssi_odoo_demo odoo.modules.registry: Registry changed, signaling through the database 
2025-09-25 07:16:08,027 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:08] "POST /web/dataset/call_button HTTP/1.1" 200 - 3865 1.605 1.451
2025-09-25 07:16:08,052 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_http: Generating routing map for key None 
2025-09-25 07:16:08,630 7388 WARNING ssi_odoo_demo odoo.addons.base.models.ir_qweb: Found deprecated directive @t-esc="title or 'Odoo'" in template 180. Replace by @t-out 
2025-09-25 07:16:08,631 7388 WARNING ssi_odoo_demo odoo.addons.base.models.ir_qweb: Found deprecated directive @t-esc='request.csrf_token(None)' in template 180. Replace by @t-out 
2025-09-25 07:16:08,631 7388 WARNING ssi_odoo_demo odoo.addons.base.models.ir_qweb: Found deprecated directive @t-esc='debug' in template 180. Replace by @t-out 
2025-09-25 07:16:08,636 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:08] "GET /web HTTP/1.1" 200 - 90 0.049 0.538
2025-09-25 07:16:08,687 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:08] "GET /web/webclient/translations/e997a8f1063632df7da47f87475348715e3f4ead?lang=en_US HTTP/1.1" 200 - 4 0.003 0.007
2025-09-25 07:16:09,464 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:09] "POST /web/dataset/call_kw/res.users/systray_get_activities HTTP/1.1" 200 - 6 0.010 0.008
2025-09-25 07:16:09,522 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:09] "POST /mail/init_messaging HTTP/1.1" 200 - 68 0.058 0.066
2025-09-25 07:16:09,534 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:09] "POST /web/action/load HTTP/1.1" 200 - 6 0.005 0.066
2025-09-25 07:16:09,550 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:09] "GET /web/manifest.webmanifest HTTP/1.1" 200 - 14 0.015 0.069
2025-09-25 07:16:09,587 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:09] "GET /web/image?model=res.users&field=avatar_128&id=6 HTTP/1.1" 304 - 22 0.046 0.080
2025-09-25 07:16:09,648 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:09] "POST /mail/load_message_failures HTTP/1.1" 200 - 3 0.001 0.006
2025-09-25 07:16:09,860 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:09] "POST /mail/inbox/messages HTTP/1.1" 200 - 11 0.013 0.012
2025-09-25 07:16:09,917 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:09] "GET /discuss/channel/1/avatar_128?unique=daf1d71dbd4db2c2a552775653d41f98f66a7823e6a81834db4e1e97cf818579349c44f421334d00fe4a9abacd6a29fb82af2b1b93a07a5eb06c7eb6b9f90623 HTTP/1.1" 200 - 12 0.007 0.016
2025-09-25 07:16:09,943 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:09] "GET /web/image/res.partner/2/avatar_128?unique=2025-09-25%2004%3A01%3A04 HTTP/1.1" 200 - 20 0.009 0.025
2025-09-25 07:16:11,467 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:11] "GET /web/service-worker.js HTTP/1.1" 200 - 1 0.000 0.005
2025-09-25 07:16:13,308 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:13] "POST /web/action/load HTTP/1.1" 200 - 8 0.005 0.008
2025-09-25 07:16:13,616 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:13] "POST /web/dataset/call_kw/stock.picking.type/get_views HTTP/1.1" 200 - 50 0.027 0.047
2025-09-25 07:16:13,678 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:13] "POST /web/dataset/call_kw/stock.picking.type/web_search_read HTTP/1.1" 200 - 36 0.024 0.013
2025-09-25 07:16:25,033 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:25] "POST /web/action/load HTTP/1.1" 200 - 10 0.003 0.011
2025-09-25 07:16:25,529 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:25] "POST /web/dataset/call_kw/purchase.order/get_views HTTP/1.1" 200 - 100 0.051 0.125
2025-09-25 07:16:25,627 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:25] "POST /web/dataset/call_kw/purchase.order/web_search_read HTTP/1.1" 200 - 14 0.013 0.013
2025-09-25 07:16:25,845 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:25] "POST /web/dataset/call_kw/res.users/has_group HTTP/1.1" 200 - 2 0.001 0.003
2025-09-25 07:16:25,912 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:25] "POST /web/dataset/call_kw/purchase.order/retrieve_dashboard HTTP/1.1" 200 - 37 0.038 0.016
2025-09-25 07:16:26,280 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:26] "GET /web/image/res.users/6/avatar_128 HTTP/1.1" 200 - 22 0.016 0.024
2025-09-25 07:16:33,430 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:33] "POST /web/dataset/call_kw/purchase.order/web_read HTTP/1.1" 200 - 151 0.134 0.073
2025-09-25 07:16:33,894 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:33] "GET /web/bundle/web_editor.backend_assets_wysiwyg?lang=en_US&debug=1 HTTP/1.1" 200 - 7 0.002 0.063
2025-09-25 07:16:34,131 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:34] "GET /web/assets/9e6b13d/web_editor.backend_assets_wysiwyg.min.css HTTP/1.1" 200 - 3 0.001 0.046
2025-09-25 07:16:34,376 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:34] "GET /web/assets/654c10b/web_editor.backend_assets_wysiwyg.min.js HTTP/1.1" 200 - 3 0.001 0.174
2025-09-25 07:16:34,443 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:34] "POST /web/dataset/call_kw/ir.ui.view/render_public_asset HTTP/1.1" 200 - 9 0.005 0.020
2025-09-25 07:16:34,849 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:34] "POST /mail/thread/data HTTP/1.1" 200 - 52 0.041 0.041
2025-09-25 07:16:34,863 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:34] "POST /mail/thread/messages HTTP/1.1" 200 - 24 0.031 0.022
2025-09-25 07:16:34,867 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:34] "GET /web/static/img/openhand.cur HTTP/1.1" 200 - 0 0.000 0.003
2025-09-25 07:16:35,062 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:35] "GET /web/image?field=avatar_128&id=7&model=res.partner&unique=2025-09-25%2004%3A01%3A04 HTTP/1.1" 200 - 20 0.012 0.023
2025-09-25 07:16:40,237 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:40] "POST /web/dataset/call_button HTTP/1.1" 200 - 21 0.005 0.018
2025-09-25 07:16:40,709 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:40] "POST /web/dataset/call_kw/stock.picking/get_views HTTP/1.1" 200 - 88 0.044 0.109
2025-09-25 07:16:40,886 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:40] "POST /web/dataset/call_kw/stock.picking/web_read HTTP/1.1" 200 - 72 0.047 0.036
2025-09-25 07:16:41,026 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:41] "POST /web/dataset/call_kw/stock.picking.type/check_access_rights HTTP/1.1" 200 - 1 0.000 0.004
2025-09-25 07:16:41,273 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:41] "POST /mail/thread/data HTTP/1.1" 200 - 44 0.025 0.028
2025-09-25 07:16:41,432 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:41] "POST /mail/thread/messages HTTP/1.1" 200 - 32 0.026 0.026
2025-09-25 07:16:41,487 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:41] "GET /web/image?field=avatar_128&id=2&model=res.partner&unique=2025-09-25%2004%3A01%3A04 HTTP/1.1" 200 - 20 0.012 0.016
2025-09-25 07:16:43,598 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:43] "POST /web/dataset/call_button HTTP/1.1" 200 - 17 0.009 0.010
2025-09-25 07:16:43,878 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:43] "POST /web/dataset/call_kw/stock.move.line/get_views HTTP/1.1" 200 - 20 0.014 0.030
2025-09-25 07:16:43,941 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:43] "POST /web/dataset/call_kw/stock.move.line/web_search_read HTTP/1.1" 200 - 27 0.014 0.018
2025-09-25 07:16:44,576 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_cron: Starting job `Base: Auto-vacuum internal data`. 
2025-09-25 07:16:44,626 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_attachment: filestore gc 0 checked, 0 removed 
2025-09-25 07:16:44,700 7388 INFO ssi_odoo_demo odoo.addons.base.models.res_users: GC'd 0 user log entries 
2025-09-25 07:16:44,734 7388 INFO ssi_odoo_demo odoo.addons.auth_totp.models.auth_totp: GC'd 0 totp devices entries 
2025-09-25 07:16:44,748 7388 INFO ssi_odoo_demo odoo.models.unlink: User #1 deleted bus.bus records with IDs: [60, 61] 
2025-09-25 07:16:44,845 7388 INFO ssi_odoo_demo odoo.addons.sms.models.sms_sms: GC'd 0 sms marked for deletion 
2025-09-25 07:16:45,006 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_cron: Job done: `Base: Auto-vacuum internal data` (0.430s). 
2025-09-25 07:16:47,249 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:47] "POST /web/action/load HTTP/1.1" 200 - 9 0.006 0.014
2025-09-25 07:16:47,679 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:47] "POST /web/dataset/call_kw/stock.picking/get_views HTTP/1.1" 200 - 32 0.018 0.086
2025-09-25 07:16:47,922 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:47] "POST /web/dataset/call_kw/stock.picking/web_read HTTP/1.1" 200 - 72 0.048 0.047
2025-09-25 07:16:48,036 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:48] "POST /mail/thread/data HTTP/1.1" 200 - 44 0.018 0.023
2025-09-25 07:16:48,264 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:48] "POST /mail/thread/messages HTTP/1.1" 200 - 7 0.006 0.006
2025-09-25 07:16:49,458 7388 WARNING ssi_odoo_demo odoo.addons.base.models.ir_qweb: Found deprecated directive @t-esc='line.product_id.display_name' in template 1032. Replace by @t-out 
2025-09-25 07:16:49,458 7388 WARNING ssi_odoo_demo odoo.addons.base.models.ir_qweb: Found deprecated directive @t-esc='line.qty_received' in template 1032. Replace by @t-out 
2025-09-25 07:16:49,460 7388 WARNING ssi_odoo_demo odoo.addons.base.models.ir_qweb: Found deprecated directive @t-esc='float(qty_received)' in template 1032. Replace by @t-out 
2025-09-25 07:16:49,493 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:49] "POST /web/dataset/call_button HTTP/1.1" 200 - 170 0.147 0.099
2025-09-25 07:16:49,837 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:49] "POST /mail/thread/messages HTTP/1.1" 200 - 24 0.015 0.019
2025-09-25 07:16:49,936 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:49] "POST /web/dataset/call_kw/stock.picking/web_read HTTP/1.1" 200 - 66 0.091 0.041
2025-09-25 07:16:50,119 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:50] "POST /mail/thread/data HTTP/1.1" 200 - 51 0.029 0.021
2025-09-25 07:16:58,861 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:58] "POST /web/dataset/call_button HTTP/1.1" 200 - 31 0.026 0.032
2025-09-25 07:16:59,247 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:59] "POST /web/dataset/call_kw/stock.valuation.layer/get_views HTTP/1.1" 200 - 31 0.021 0.050
2025-09-25 07:16:59,442 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:59] "POST /web/dataset/call_kw/stock.valuation.layer/web_read_group HTTP/1.1" 200 - 11 0.008 0.008
2025-09-25 07:16:59,608 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:16:59] "POST /web/dataset/call_kw/product.product/web_read HTTP/1.1" 200 - 44 0.026 0.024
2025-09-25 07:17:02,205 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:02] "POST /web/dataset/call_kw/stock.valuation.layer/web_search_read HTTP/1.1" 200 - 13 0.009 0.011
2025-09-25 07:17:04,426 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:04] "POST /web/dataset/call_kw/stock.picking/web_read HTTP/1.1" 200 - 66 0.047 0.034
2025-09-25 07:17:04,672 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:04] "POST /mail/thread/data HTTP/1.1" 200 - 51 0.031 0.038
2025-09-25 07:17:04,765 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:04] "POST /mail/thread/messages HTTP/1.1" 200 - 7 0.006 0.008
2025-09-25 07:17:05,216 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:05] "POST /web/dataset/call_button HTTP/1.1" 200 - 16 0.008 0.010
2025-09-25 07:17:05,585 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:05] "POST /web/dataset/call_kw/stock.move.line/web_search_read HTTP/1.1" 200 - 26 0.026 0.024
2025-09-25 07:17:14,903 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:14] "POST /web/action/load HTTP/1.1" 200 - 10 0.006 0.011
2025-09-25 07:17:15,498 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:15] "POST /web/dataset/call_kw/account.move/get_views HTTP/1.1" 200 - 102 0.056 0.226
2025-09-25 07:17:15,516 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:15] "POST /web/dataset/call_kw/account.move/web_search_read HTTP/1.1" 200 - 7 0.003 0.007
2025-09-25 07:17:16,001 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:16] "POST /onboarding/account_invoice HTTP/1.1" 200 - 63 0.058 0.092
2025-09-25 07:17:16,097 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:16] "GET /web/static/img/smiling_face.svg HTTP/1.1" 200 - 0 0.000 0.002
2025-09-25 07:17:16,348 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:16] "GET /web/image/onboarding.onboarding.step/2/step_image HTTP/1.1" 200 - 5 0.002 0.027
2025-09-25 07:17:16,370 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:16] "GET /web/image/onboarding.onboarding.step/3/step_image HTTP/1.1" 200 - 5 0.006 0.023
2025-09-25 07:17:16,370 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:16] "GET /web/image/onboarding.onboarding.step/11/step_image HTTP/1.1" 200 - 5 0.006 0.020
2025-09-25 07:17:16,384 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:16] "GET /web/image/onboarding.onboarding.step/4/step_image HTTP/1.1" 200 - 5 0.020 0.022
2025-09-25 07:17:21,199 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:21] "POST /web/dataset/call_kw/account.move/web_search_read HTTP/1.1" 200 - 7 0.005 0.008
2025-09-25 07:17:21,644 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:21] "POST /onboarding/account_invoice HTTP/1.1" 200 - 61 0.040 0.084
2025-09-25 07:17:23,414 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:23] "POST /web/action/load HTTP/1.1" 200 - 9 0.004 0.009
2025-09-25 07:17:23,873 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:23] "POST /web/dataset/call_kw/account.payment/get_views HTTP/1.1" 200 - 53 0.027 0.116
2025-09-25 07:17:24,020 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:24] "POST /web/dataset/call_kw/account.payment/web_search_read HTTP/1.1" 200 - 11 0.024 0.010
2025-09-25 07:17:29,934 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:29] "POST /web/dataset/call_kw/stock.picking.type/web_search_read HTTP/1.1" 200 - 36 0.016 0.022
2025-09-25 07:17:39,895 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:39] "POST /web/dataset/call_kw/purchase.order/web_search_read HTTP/1.1" 200 - 14 0.009 0.008
2025-09-25 07:17:40,178 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:40] "POST /web/dataset/call_kw/purchase.order/retrieve_dashboard HTTP/1.1" 200 - 36 0.017 0.019
2025-09-25 07:17:45,494 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:45] "POST /web/dataset/call_kw/purchase.order/web_read HTTP/1.1" 200 - 156 0.096 0.050
2025-09-25 07:17:45,931 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:45] "POST /mail/thread/data HTTP/1.1" 200 - 52 0.039 0.032
2025-09-25 07:17:45,941 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:45] "POST /mail/thread/messages HTTP/1.1" 200 - 22 0.016 0.016
2025-09-25 07:17:50,352 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:50] "POST /web/dataset/call_button HTTP/1.1" 200 - 275 0.250 0.251
2025-09-25 07:17:50,831 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:50] "POST /web/dataset/call_kw/account.move/get_views HTTP/1.1" 200 - 29 0.023 0.150
2025-09-25 07:17:51,062 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:51] "POST /web/dataset/call_kw/account.move/web_read HTTP/1.1" 200 - 113 0.094 0.057
2025-09-25 07:17:51,264 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:51] "POST /mail/thread/data HTTP/1.1" 200 - 52 0.039 0.037
2025-09-25 07:17:51,664 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:51] "POST /mail/thread/messages HTTP/1.1" 200 - 22 0.018 0.025
2025-09-25 07:17:53,933 7388 WARNING ssi_odoo_demo odoo.http: The Bill/Refund date is required to validate this document. 
2025-09-25 07:17:53,933 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:17:53] "POST /web/dataset/call_button HTTP/1.1" 200 - 42 0.024 0.115
2025-09-25 07:18:01,098 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:01] "POST /web/dataset/call_kw/account.move/onchange HTTP/1.1" 200 - 114 0.077 0.085
2025-09-25 07:18:02,511 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:02] "POST /web/dataset/call_kw/account.move/web_save HTTP/1.1" 200 - 132 0.079 0.119
2025-09-25 07:18:02,832 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:02] "POST /mail/thread/messages HTTP/1.1" 200 - 7 0.007 0.006
2025-09-25 07:18:02,927 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:02] "POST /mail/thread/data HTTP/1.1" 200 - 52 0.067 0.030
2025-09-25 07:18:03,063 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:03] "POST /web/dataset/call_button HTTP/1.1" 200 - 153 0.100 0.132
2025-09-25 07:18:03,095 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:03] "POST /mail/thread/messages HTTP/1.1" 200 - 24 0.011 0.016
2025-09-25 07:18:03,531 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:03] "POST /web/dataset/call_kw/account.move/web_read HTTP/1.1" 200 - 124 0.079 0.074
2025-09-25 07:18:03,633 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:03] "POST /mail/thread/data HTTP/1.1" 200 - 57 0.045 0.044
2025-09-25 07:18:06,111 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:06] "POST /web/dataset/call_button HTTP/1.1" 200 - 11 0.008 0.007
2025-09-25 07:18:06,402 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:06] "POST /web/dataset/call_kw/account.payment.register/get_views HTTP/1.1" 200 - 16 0.010 0.045
2025-09-25 07:18:06,484 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:06] "POST /web/dataset/call_kw/account.payment.register/onchange HTTP/1.1" 200 - 38 0.035 0.025
2025-09-25 07:18:08,696 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:08] "POST /web/dataset/call_kw/account.journal/name_search HTTP/1.1" 200 - 8 0.006 0.009
2025-09-25 07:18:09,735 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:09] "POST /web/dataset/call_kw/account.payment.register/onchange HTTP/1.1" 200 - 32 0.010 0.034
2025-09-25 07:18:10,761 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:10] "POST /web/dataset/call_kw/account.payment.register/web_save HTTP/1.1" 200 - 50 0.044 0.031
2025-09-25 07:18:11,385 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:11] "POST /web/dataset/call_button HTTP/1.1" 200 - 233 0.154 0.156
2025-09-25 07:18:11,427 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:11] "POST /mail/thread/messages HTTP/1.1" 200 - 24 0.017 0.020
2025-09-25 07:18:11,842 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:11] "POST /web/dataset/call_kw/account.move/web_read HTTP/1.1" 200 - 126 0.083 0.061
2025-09-25 07:18:12,220 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:12] "POST /mail/thread/data HTTP/1.1" 200 - 57 0.036 0.021
2025-09-25 07:18:14,609 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:14] "POST /web/dataset/call_button HTTP/1.1" 200 - 30 0.023 0.019
2025-09-25 07:18:14,983 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:14] "POST /web/dataset/call_kw/purchase.order/get_views HTTP/1.1" 200 - 14 0.010 0.055
2025-09-25 07:18:15,334 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:15] "POST /web/dataset/call_kw/purchase.order/web_read HTTP/1.1" 200 - 156 0.094 0.063
2025-09-25 07:18:15,470 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:15] "POST /mail/thread/data HTTP/1.1" 200 - 52 0.043 0.039
2025-09-25 07:18:15,761 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:15] "POST /mail/thread/messages HTTP/1.1" 200 - 7 0.010 0.008
2025-09-25 07:18:18,836 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:18] "POST /web/dataset/call_button HTTP/1.1" 200 - 20 0.009 0.013
2025-09-25 07:18:19,204 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:19] "POST /web/dataset/call_kw/account.move/web_read HTTP/1.1" 200 - 126 0.063 0.073
2025-09-25 07:18:19,350 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:19] "POST /mail/thread/data HTTP/1.1" 200 - 57 0.050 0.044
2025-09-25 07:18:19,621 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:19] "POST /mail/thread/messages HTTP/1.1" 200 - 7 0.006 0.004
2025-09-25 07:18:20,768 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:20] "POST /web/dataset/call_button HTTP/1.1" 200 - 28 0.016 0.016
2025-09-25 07:18:21,186 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:21] "POST /web/dataset/call_kw/purchase.order/web_read HTTP/1.1" 200 - 156 0.111 0.089
2025-09-25 07:18:21,320 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:21] "POST /mail/thread/data HTTP/1.1" 200 - 52 0.035 0.044
2025-09-25 07:18:21,611 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:21] "POST /mail/thread/messages HTTP/1.1" 200 - 7 0.006 0.008
2025-09-25 07:18:22,276 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:22] "POST /web/dataset/call_button HTTP/1.1" 200 - 19 0.013 0.018
2025-09-25 07:18:22,570 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:22] "POST /web/dataset/call_kw/stock.picking/web_read HTTP/1.1" 200 - 66 0.039 0.031
2025-09-25 07:18:22,681 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:22] "POST /mail/thread/data HTTP/1.1" 200 - 51 0.049 0.034
2025-09-25 07:18:22,936 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:22] "POST /mail/thread/messages HTTP/1.1" 200 - 7 0.003 0.006
2025-09-25 07:18:24,428 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:24] "POST /web/dataset/call_button HTTP/1.1" 200 - 16 0.015 0.009
2025-09-25 07:18:24,693 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:24] "POST /web/dataset/call_kw/product.label.layout/get_views HTTP/1.1" 200 - 13 0.005 0.031
2025-09-25 07:18:24,752 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:24] "POST /web/dataset/call_kw/product.label.layout/onchange HTTP/1.1" 200 - 11 0.006 0.010
2025-09-25 07:18:34,188 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:34] "POST /web/dataset/call_kw/product.pricelist/name_search HTTP/1.1" 200 - 9 0.007 0.008
2025-09-25 07:18:39,331 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:39] "POST /web/dataset/call_kw/product.label.layout/web_save HTTP/1.1" 200 - 21 0.028 0.011
2025-09-25 07:18:39,607 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:39] "POST /web/dataset/call_button HTTP/1.1" 200 - 52 0.042 0.012
2025-09-25 07:18:39,646 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:39] "POST /report/check_wkhtmltopdf HTTP/1.1" 200 - 1 0.000 0.004
2025-09-25 07:18:39,933 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:39] "GET /web/static/img/spin.svg HTTP/1.1" 200 - 0 0.000 0.019
2025-09-25 07:18:45,241 7388 WARNING ssi_odoo_demo odoo.addons.base.models.ir_qweb: Found deprecated directive @t-esc="title or 'Odoo Report'" in template 192. Replace by @t-out 
2025-09-25 07:18:45,404 7388 WARNING ssi_odoo_demo odoo.addons.base.models.ir_qweb: Found deprecated directive @t-esc="title or 'Odoo Report'" in template 192. Replace by @t-out 
2025-09-25 07:18:48,460 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:48] "GET /web/assets/3b99e7d/web.report_assets_common.min.css HTTP/1.1" 200 - 3 0.010 0.060
2025-09-25 07:18:48,527 7388 INFO ssi_odoo_demo odoo.addons.base.models.assetsbundle: Generating a new asset bundle attachment /web/assets/1561b26/web.report_assets_pdf.min.css (id:479) 
2025-09-25 07:18:48,564 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:48] "GET /web/assets/1561b26/web.report_assets_pdf.min.css HTTP/1.1" 200 - 11 0.030 0.145
2025-09-25 07:18:49,760 7388 INFO ssi_odoo_demo odoo.addons.base.models.assetsbundle: Generating a new asset bundle attachment /web/assets/d8af00c/web.report_assets_common.min.js (id:480) 
2025-09-25 07:18:50,056 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:50] "GET /web/assets/d8af00c/web.report_assets_common.min.js HTTP/1.1" 200 - 15 0.027 1.639
2025-09-25 07:18:50,280 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:50] "GET /web/static/fonts/lato/Lato-Reg-webfont.woff HTTP/1.1" 200 - 0 0.000 0.026
2025-09-25 07:18:50,304 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:50] "GET /web/static/fonts/lato/Lato-Hai-webfont.woff HTTP/1.1" 200 - 0 0.000 0.036
2025-09-25 07:18:50,307 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:50] "GET /web/static/fonts/lato/Lato-Lig-webfont.woff HTTP/1.1" 200 - 0 0.000 0.041
2025-09-25 07:18:52,336 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:52] "GET /web/static/fonts/lato/Lato-Bol-webfont.woff HTTP/1.1" 200 - 0 0.000 0.033
2025-09-25 07:18:52,344 7388 INFO ? werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:52] "GET /web/static/fonts/lato/Lato-Bla-webfont.woff HTTP/1.1" 200 - 0 0.000 0.025
2025-09-25 07:18:57,674 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:57] "POST /report/download HTTP/1.1" 200 - 1218 1.365 16.381
2025-09-25 07:18:57,704 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:57] "POST /mail/thread/messages HTTP/1.1" 200 - 7 0.006 0.006
2025-09-25 07:18:58,129 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:58] "POST /web/dataset/call_kw/stock.picking/web_read HTTP/1.1" 200 - 66 0.050 0.076
2025-09-25 07:18:58,647 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:18:58] "POST /mail/thread/data HTTP/1.1" 200 - 51 0.099 0.057
2025-09-25 07:19:05,509 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:19:05] "POST /web/dataset/call_kw/purchase.order/web_search_read HTTP/1.1" 200 - 14 0.006 0.012
2025-09-25 07:19:05,867 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:19:05] "POST /web/dataset/call_kw/purchase.order/retrieve_dashboard HTTP/1.1" 200 - 36 0.022 0.014
2025-09-25 07:19:30,884 7388 WARNING ssi_odoo_demo odoo.addons.base.models.ir_qweb: Found deprecated directive @t-esc="title or 'Odoo'" in template 180. Replace by @t-out 
2025-09-25 07:19:30,885 7388 WARNING ssi_odoo_demo odoo.addons.base.models.ir_qweb: Found deprecated directive @t-esc='request.csrf_token(None)' in template 180. Replace by @t-out 
2025-09-25 07:19:30,886 7388 WARNING ssi_odoo_demo odoo.addons.base.models.ir_qweb: Found deprecated directive @t-esc='debug' in template 180. Replace by @t-out 
2025-09-25 07:19:30,892 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:19:30] "GET /web HTTP/1.1" 200 - 43 0.022 0.380
2025-09-25 07:19:31,078 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:19:31] "POST /web/action/load HTTP/1.1" 200 - 10 0.007 0.011
2025-09-25 07:19:31,164 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:19:31] "POST /mail/init_messaging HTTP/1.1" 200 - 61 0.047 0.045
2025-09-25 07:19:31,434 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:19:31] "POST /web/dataset/call_kw/res.users/systray_get_activities HTTP/1.1" 200 - 4 0.006 0.005
2025-09-25 07:19:31,478 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:19:31] "POST /mail/load_message_failures HTTP/1.1" 200 - 3 0.003 0.007
2025-09-25 07:19:31,512 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:19:31] "GET /web/image?model=res.users&field=avatar_128&id=6 HTTP/1.1" 304 - 22 0.014 0.021
2025-09-25 07:19:31,606 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:19:31] "POST /web/dataset/call_kw/purchase.order/get_views HTTP/1.1" 200 - 38 0.021 0.142
2025-09-25 07:19:31,855 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:19:31] "POST /web/dataset/call_kw/purchase.order/web_search_read HTTP/1.1" 200 - 14 0.013 0.013
2025-09-25 07:19:31,926 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:19:31] "POST /web/dataset/call_kw/res.users/has_group HTTP/1.1" 200 - 1 0.001 0.004
2025-09-25 07:19:31,939 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:19:31] "GET /web/manifest.webmanifest HTTP/1.1" 200 - 13 0.005 0.013
2025-09-25 07:19:32,303 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:19:32] "POST /web/dataset/call_kw/purchase.order/retrieve_dashboard HTTP/1.1" 200 - 36 0.030 0.024
2025-09-25 07:19:32,536 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:19:32] "GET /web/image/res.users/6/avatar_128 HTTP/1.1" 304 - 22 0.009 0.018
2025-09-25 07:19:33,814 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:19:33] "GET /web/service-worker.js HTTP/1.1" 200 - 1 0.001 0.003
2025-09-25 07:19:38,903 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:19:38] "POST /web/action/load HTTP/1.1" 200 - 10 0.008 0.005
2025-09-25 07:19:39,266 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:19:39] "POST /web/dataset/call_kw/purchase.order/get_views HTTP/1.1" 200 - 39 0.024 0.094
2025-09-25 07:19:39,289 7388 INFO ssi_odoo_demo werkzeug: 127.0.0.1 - - [25/Sep/2025 07:19:39] "POST /web/dataset/call_kw/purchase.order/web_search_read HTTP/1.1" 200 - 14 0.009 0.008
2025-09-25 07:24:45,092 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_cron: Starting job `Notification: Send scheduled message notifications`. 
2025-09-25 07:24:45,095 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_cron: Job done: `Notification: Send scheduled message notifications` (0.003s). 
2025-09-25 07:24:45,111 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_cron: Starting job `Partner Autocomplete: Sync with remote DB`. 
2025-09-25 07:24:45,114 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_cron: Job done: `Partner Autocomplete: Sync with remote DB` (0.003s). 
2025-09-25 07:24:45,119 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_cron: Starting job `SMS: SMS Queue Manager`. 
2025-09-25 07:24:45,123 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_cron: Job done: `SMS: SMS Queue Manager` (0.004s). 
2025-09-25 07:24:45,129 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_cron: Starting job `Snailmail: process letters queue`. 
2025-09-25 07:24:45,133 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_cron: Job done: `Snailmail: process letters queue` (0.004s). 
2025-09-25 07:24:45,139 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_cron: Starting job `Mail: Email Queue Manager`. 
2025-09-25 07:24:49,249 7388 INFO ssi_odoo_demo odoo.addons.base.models.ir_cron: Job done: `Mail: Email Queue Manager` (4.109s). 
2025-09-25 07:27:28,719 7388 INFO ssi_odoo_demo odoo.service.server: Initiating shutdown 
2025-09-25 07:27:28,719 7388 INFO ssi_odoo_demo odoo.service.server: Hit CTRL-C again or send a second signal to force the shutdown. 
2025-09-25 07:27:29,112 7388 INFO ssi_odoo_demo odoo.sql_db: ConnectionPool(used=0/count=0/max=64): Closed 6 connections  
